<!DOCTYPE html>
<html lang="en"><head>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-html/tabby.min.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/light-border.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-html.min.css" rel="stylesheet" data-mode="light">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/quarto-contrib/roughnotation-0.5.1/rough-notation.iife.js"></script>
<script src="../site_libs/quarto-contrib/roughnotation-init-1.0.0/rough.js"></script><meta charset="utf-8">
  <meta name="generator" content="quarto-1.5.57">

  <title>Analisi di dati di trascrittomica con R – m3_rnaseq_pipeline</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="../site_libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="../site_libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
  </style>
  <link rel="stylesheet" href="../site_libs/revealjs/dist/theme/quarto.css">
  <link rel="stylesheet" href="css/theme.css">
  <link rel="stylesheet" href="css/custom.css">
  <link href="../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="../site_libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="../site_libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="../site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">

  .callout {
    margin-top: 1em;
    margin-bottom: 1em;  
    border-radius: .25rem;
  }

  .callout.callout-style-simple { 
    padding: 0em 0.5em;
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
    display: flex;
  }

  .callout.callout-style-default {
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
  }

  .callout .callout-body-container {
    flex-grow: 1;
  }

  .callout.callout-style-simple .callout-body {
    font-size: 1rem;
    font-weight: 400;
  }

  .callout.callout-style-default .callout-body {
    font-size: 0.9rem;
    font-weight: 400;
  }

  .callout.callout-titled.callout-style-simple .callout-body {
    margin-top: 0.2em;
  }

  .callout:not(.callout-titled) .callout-body {
      display: flex;
  }

  .callout:not(.no-icon).callout-titled.callout-style-simple .callout-content {
    padding-left: 1.6em;
  }

  .callout.callout-titled .callout-header {
    padding-top: 0.2em;
    margin-bottom: -0.2em;
  }

  .callout.callout-titled .callout-title  p {
    margin-top: 0.5em;
    margin-bottom: 0.5em;
  }
    
  .callout.callout-titled.callout-style-simple .callout-content  p {
    margin-top: 0;
  }

  .callout.callout-titled.callout-style-default .callout-content  p {
    margin-top: 0.7em;
  }

  .callout.callout-style-simple div.callout-title {
    border-bottom: none;
    font-size: .9rem;
    font-weight: 600;
    opacity: 75%;
  }

  .callout.callout-style-default  div.callout-title {
    border-bottom: none;
    font-weight: 600;
    opacity: 85%;
    font-size: 0.9rem;
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-default div.callout-content {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-simple .callout-icon::before {
    height: 1rem;
    width: 1rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 1rem 1rem;
  }

  .callout.callout-style-default .callout-icon::before {
    height: 0.9rem;
    width: 0.9rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 0.9rem 0.9rem;
  }

  .callout-title {
    display: flex
  }
    
  .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  .callout.no-icon::before {
    display: none !important;
  }

  .callout.callout-titled .callout-body > .callout-content > :last-child {
    padding-bottom: 0.5rem;
    margin-bottom: 0;
  }

  .callout.callout-titled .callout-icon::before {
    margin-top: .5rem;
    padding-right: .5rem;
  }

  .callout:not(.callout-titled) .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  /* Callout Types */

  div.callout-note {
    border-left-color: #4582ec !important;
  }

  div.callout-note .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEU0lEQVRYCcVXTWhcVRQ+586kSUMMxkyaElstCto2SIhitS5Ek8xUKV2poatCcVHtUlFQk8mbaaziwpWgglJwVaquitBOfhQXFlqlzSJpFSpIYyXNjBNiTCck7x2/8/LeNDOZxDuEkgOXe++553zfefee+/OYLOXFk3+1LLrRdiO81yNqZ6K9cG0P3MeFaMIQjXssE8Z1JzLO9ls20MBZX7oG8w9GxB0goaPrW5aNMp1yOZIa7Wv6o2ykpLtmAPs/vrG14Z+6d4jpbSKuhdcSyq9wGMPXjonwmESXrriLzFGOdDBLB8Y6MNYBu0dRokSygMA/mrun8MGFN3behm6VVAwg4WR3i6FvYK1T7MHo9BK7ydH+1uurECoouk5MPRyVSBrBHMYwVobG2aOXM07sWrn5qgB60rc6mcwIDJtQrnrEr44kmy+UO9r0u9O5/YbkS9juQckLed3DyW2XV/qWBBB3ptvI8EUY3I9p/67OW+g967TNr3Sotn3IuVlfMLVnsBwH4fsnebJvyGm5GeIUA3jljERmrv49SizPYuq+z7c2H/jlGC+Ghhupn/hcapqmcudB9jwJ/3jvnvu6vu5lVzF1fXyZuZZ7U8nRmVzytvT+H3kilYvH09mLWrQdwFSsFEsxFVs5fK7A0g8gMZjbif4ACpKbjv7gNGaD8bUrlk8x+KRflttr22JEMRUbTUwwDQScyzPgedQHZT0xnx7ujw2jfVfExwYHwOsDTjLdJ2ebmeQIlJ7neo41s/DrsL3kl+W2lWvAga0tR3zueGr6GL78M3ifH0rGXrBC2aAR8uYcIA5gwV8zIE8onoh8u0Fca/ciF7j1uOzEnqcIm59sEXoGc0+z6+H45V1CvAvHcD7THztu669cnp+L0okAeIc6zjbM/24LgGM1gZk7jnRu1aQWoU9sfUOuhrmtaPIO3YY1KLLWZaEO5TKUbMY5zx8W9UJ6elpLwKXbsaZ4EFl7B4bMtDv0iRipKoDQT2sNQI9b1utXFdYisi+wzZ/ri/1m7QfDgEuvgUUEIJPq3DhX/5DWNqIXDOweC2wvIR90Oq3lDpdMIgD2r0dXvGdsEW5H6x6HLRJYU7C69VefO1x8Gde1ZFSJLfWS1jbCnhtOPxmpfv2LXOA2Xk2tvnwKKPFuZ/oRmwBwqRQDcKNeVQkYcOjtWVBuM/JuYw5b6isojIkYxyYAFn5K7ZBF10fea52y8QltAg6jnMqNHFBmGkQ1j+U43HMi2xMar1Nv0zGsf1s8nUsmUtPOOrbFIR8bHFDMB5zL13Gmr/kGlCkUzedTzzmzsaJXhYawnA3UmARpiYj5ooJZiUoxFRtK3X6pgNPv+IZVPcnwbOl6f+aBaO1CNvPW9n9LmCp01nuSaTRF2YxHqZ8DYQT6WsXT+RD6eUztwYLZ8rM+rcPxamv1VQzFUkzFXvkiVrySGQgJNvXHJAxiU3/NwiC03rSf05VBaPtu/Z7/B8Yn/w7eguloAAAAAElFTkSuQmCC');
  }

  div.callout-note.callout-style-default .callout-title {
    background-color: #dae6fb
  }

  div.callout-important {
    border-left-color: #d9534f !important;
  }

  div.callout-important .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEKklEQVRYCcVXTWhcVRS+575MJym48A+hSRFr00ySRQhURRfd2HYjk2SSTokuBCkU2o0LoSKKraKIBTcuFCoidGFD08nkBzdREbpQ1EDNIv8qSGMFUboImMSZd4/f9zJv8ibJMC8xJQfO3HPPPef7zrvvvnvviIkpC9nsw0UttFunbUhpFzFtarSd6WJkStVMw5xyVqYTvkwfzuf/5FgtkVoB0729j1rjXwThS7Vio+Mo6DNnvLfahoZ+i/o32lULuJ3NNiz7q6+pyAUkJaFF6JwaM2lUJlV0MlnQn5aTRbEu0SEqHUa0A4AdiGuB1kFXRfVyg5d87+Dg4DL6m2TLAub60ilj7A1Ec4odSAc8X95sHh7+ZRPCFo6Fnp7HfU/fBng/hi10CjCnWnJjsxvDNxWw0NfV6Rv5GgP3I3jGWXumdTD/3cbEOP2ZbOZp69yniG3FQ9z1jD7bnBu9Fc2tKGC2q+uAJOQHBDRiZX1x36o7fWBs7J9ownbtO+n0/qWkvW7UPIfc37WgT6ZGR++EOJyeQDSb9UB+DZ1G6DdLDzyS+b/kBCYGsYgJbSQHuThGKRcw5xdeQf8YdNHsc6ePXrlSYMBuSIAFTGAtQo+VuALo4BX83N190NWZWbynBjhOHsmNfFWLeL6v+ynsA58zDvvAC8j5PkbOcXCMg2PZFk3q8MjI7WAG/Dp9AwP7jdGBOOQkAvlFUB+irtm16I1Zw9YBcpGTGXYmk3kQIC/Cds55l+iMI3jqhjAuaoe+am2Jw5GT3Nbz3CkE12NavmzN5+erJW7046n/CH1RO/RVa8lBLozXk9uqykkGAyRXLWlLv5jyp4RFsG5vGVzpDLnIjTWgnRy2Rr+tDKvRc7Y8AyZq10jj8DqXdnIRNtFZb+t/ZRtXcDiVnzpqx8mPcDWxgARUqx0W1QB9MeUZiNrV4qP+Ehc+BpNgATsTX8ozYKL2NtFYAHc84fG7ndxUPr+AR/iQSns7uSUufAymwDOb2+NjK27lEFocm/EE2WpyIy/Hi66MWuMKJn8RvxIcj87IM5Vh9663ziW36kR0HNenXuxmfaD8JC7tfKbrhFr7LiZCrMjrzTeGx+PmkosrkNzW94ObzwocJ7A1HokLolY+AvkTiD/q1H0cN48c5EL8Crkttsa/AXQVDmutfyku0E7jShx49XqV3MFK8IryDhYVbj7Sj2P2eBxwcXoe8T8idsKKPRcnZw1b+slFTubwUwhktrfnAt7J++jwQtLZcm3sr9LQrjRzz6cfMv9aLvgmnAGvpoaGLxM4mAEaLV7iAzQ3oU0IvD5x9ix3yF2RAAuYAOO2f7PEFWCXZ4C9Pb2UsgDeVnFSpbFK7/IWu7TPTvBqzbGdCHOJQSxiEjt6IyZmxQyEJHv6xyQsYk//moVFsN2zP6fRImjfq7/n/wFDguUQFNEwugAAAABJRU5ErkJggg==');
  }

  div.callout-important.callout-style-default .callout-title {
    background-color: #f7dddc
  }

  div.callout-warning {
    border-left-color: #f0ad4e !important;
  }

  div.callout-warning .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAETklEQVRYCeVWW2gcVRg+58yaTUnizqbipZeX4uWhBEniBaoUX1Ioze52t7sRq6APio9V9MEaoWlVsFasRq0gltaAPuxms8lu0gcviE/FFOstVbSIxgcv6SU7EZqmdc7v9+9mJtNks51NTUH84ed889/PP+cmxP+d5FIbMJmNbpREu4WUkiTtCicKny0l1pIKmBzovF2S+hIJHX8iEu3hZJ5lNZGqyRrGSIQpq15AzF28jgpeY6yk6GVdrfFqdrD6Iw+QlB8g0YS2g7dyQmXM/IDhBhT0UCiRf59lfqmmDvzRt6kByV/m4JjtzuaujMUM2c5Z2d6JdKrRb3K2q6mA+oYVz8JnDdKPmmNthzkAk/lN63sYPgevrguc72aZX/L9C6x09GYyxBgCX4NlvyGUHOKELlm5rXeR1kchuChJt4SSwyddZRXgvwMGvYo4QSlk3/zkHD8UHxwVJA6zjZZqP8v8kK8OWLnIZtLyCAJagYC4rTGW/9Pqj92N/c+LUaAj27movwbi19tk/whRCIE7Q9vyI6yvRpftAKVTdUjOW40X3h5OXsKCdmFcx0xlLJoSuQngnrJe7Kcjm4OMq9FlC7CMmScQANuNvjfP3PjGXDBaUQmbp296S5L4DrpbrHN1T87ZVEZVCzg1FF0Ft+dKrlLukI+/c9ENo+TvlTDbYFvuKPtQ9+l052rXrgKoWkDAFnvh0wTOmYn8R5f4k/jN/fZiCM1tQx9jQQ4ANhqG4hiL0qIFTGViG9DKB7GYzgubnpofgYRwO+DFjh0Zin2m4b/97EDkXkc+f6xYAPX0KK2I/7fUQuwzuwo/L3AkcjugPNixC8cHf0FyPjWlItmLxWw4Ou9YsQCr5fijMGoD/zpdRy95HRysyXA74MWOnscpO4j2y3HAVisw85hX5+AFBRSHt4ShfLFkIMXTqyKFc46xdzQM6XbAi702a7sy04J0+feReMFKp5q9esYLCqAZYw/k14E/xcLLsFElaornTuJB0svMuJINy8xkIYuL+xPAlWRceH6+HX7THJ0djLUom46zREu7tTkxwmf/FdOZ/sh6Q8qvEAiHpm4PJ4a/doJe0gH1t+aHRgCzOvBvJedEK5OFE5jpm4AGP2a8Dxe3gGJ/pAutug9Gp6he92CsSsWBaEcxGx0FHytmIpuqGkOpldqNYQK8cSoXvd+xLxXADw0kf6UkJNFtdo5MOgaLjiQOQHcn+A6h5NuL2s0qsC2LOM75PcF3yr5STuBSAcGG+meA14K/CI21HcS4LBT6tv0QAh8Dr5l93AhZzG5ZJ4VxAqdZUEl9z7WJ4aN+svMvwHHL21UKTd1mqvChH7/Za5xzXBBKrUcB0TQ+Ulgkfbi/H/YT5EptrGzsEK7tR1B7ln9BBwckYfMiuSqklSznIuoIIOM42MQO+QnduCoFCI0bpkzjCjddHPN/F+2Yu+sd9bKNpVwHhbS3LluK/0zgfwD0xYI5dXuzlQAAAABJRU5ErkJggg==');
  }

  div.callout-warning.callout-style-default .callout-title {
    background-color: #fcefdc
  }

  div.callout-tip {
    border-left-color: #02b875 !important;
  }

  div.callout-tip .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAADr0lEQVRYCe1XTWgTQRj9ZjZV8a9SPIkKgj8I1bMHsUWrqYLVg4Ue6v9BwZOxSYsIerFao7UiUryIqJcqgtpimhbBXoSCVxUFe9CTiogUrUp2Pt+3aUI2u5vdNh4dmMzOzHvvezuz8xNFM0mjnbXaNu1MvFWRXkXEyE6aYOYJpdW4IXuA4r0fo8qqSMDBU0v1HJUgVieAXxzCsdE/YJTdFcVIZQNMyhruOMJKXYFoLfIfIvVIMWdsrd+Rpd86ZmyzzjJmLStqRn0v8lzkb4rVIXvnpScOJuAn2ACC65FkPzEdEy4TPWRLJ2h7z4cArXzzaOdKlbOvKKX25Wl00jSnrwVxAg3o4dRxhO13RBSdNvH0xSARv3adTXbBdTf64IWO2vH0LT+cv4GR1DJt+DUItaQogeBX/chhbTBxEiZ6gftlDNXTrvT7co4ub5A6gp9HIcHvzTa46OS5fBeP87Qm0fQkr4FsYgVQ7Qg+ZayaDg9jhg1GkWj8RG6lkeSacrrHgDaxdoBiZPg+NXV/KifMuB6//JmYH4CntVEHy/keA6x4h4CU5oFy8GzrBS18cLJMXcljAKB6INjWsRcuZBWVaS3GDrqB7rdapVIeA+isQ57Eev9eCqzqOa81CY05VLd6SamW2wA2H3SiTbnbSxmzfp7WtKZkqy4mdyAlGx7ennghYf8voqp9cLSgKdqNfa6RdRsAAkPwRuJZNbpByn+RrJi1RXTwdi8RQF6ymDwGMAtZ6TVE+4uoKh+MYkcLsT0Hk8eAienbiGdjJHZTpmNjlbFJNKDVAp2fJlYju6IreQxQ08UJDNYdoLSl6AadO+fFuCQqVMB1NJwPm69T04Wv5WhfcWyfXQB+wXRs1pt+nCknRa0LVzSA/2B+a9+zQJadb7IyyV24YAxKp2Jqs3emZTuNnKxsah+uabKbMk7CbTgJx/zIgQYErIeTKRQ9yD9wxVof5YolPHqaWo7TD6tJlh7jQnK5z2n3+fGdggIOx2kaa2YI9QWarc5Ce1ipNWMKeSG4DysFF52KBmTNMmn5HqCFkwy34rDg05gDwgH3bBi+sgFhN/e8QvRn8kbamCOhgrZ9GJhFDgfcMHzFb6BAtjKpFhzTjwv1KCVuxHvCbsSiEz4CANnj84cwHdFXAbAOJ4LTSAawGWFn5tDhLMYz6nWeU2wJfIhmIJBefcd/A5FWQWGgrWzyORZ3Q6HuV+Jf0Bj+BTX69fm1zWgK7By1YTXchFDORywnfQ7GpzOo6S+qECrsx2ifVQAAAABJRU5ErkJggg==');
  }

  div.callout-tip.callout-style-default .callout-title {
    background-color: #ccf1e3
  }

  div.callout-caution {
    border-left-color: #fd7e14 !important;
  }

  div.callout-caution .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAACV0lEQVRYCdVWzWoUQRCuqp2ICBLJXgITZL1EfQDBW/bkzUMUD7klD+ATSHBEfAIfQO+iXsWDxJsHL96EHAwhgzlkg8nBg25XWb0zIb0zs9muYYWkoKeru+vn664fBqElyZNuyh167NXJ8Ut8McjbmEraKHkd7uAnAFku+VWdb3reSmRV8PKSLfZ0Gjn3a6Xlcq9YGb6tADjn+lUfTXtVmaZ1KwBIvFI11rRXlWlatwIAAv2asaa9mlB9wwygiDX26qaw1yYPzFXg2N1GgG0FMF8Oj+VIx7E/03lHx8UhvYyNZLN7BwSPgekXXLribw7w5/c8EF+DBK5idvDVYtEEwMeYefjjLAdEyQ3M9nfOkgnPTEkYU+sxMq0BxNR6jExrAI31H1rzvLEfRIdgcv1XEdj6QTQAS2wtstEALLG1yEZ3QhH6oDX7ExBSFEkFINXH98NTrme5IOaaA7kIfiu2L8A3qhH9zRbukdCqdsA98TdElyeMe5BI8Rs2xHRIsoTSSVFfCFCWGPn9XHb4cdobRIWABNf0add9jakDjQJpJ1bTXOJXnnRXHRf+dNL1ZV1MBRCXhMbaHqGI1JkKIL7+i8uffuP6wVQAzO7+qVEbF6NbS0LJureYcWXUUhH66nLR5rYmva+2tjRFtojkM2aD76HEGAD3tPtKM309FJg5j/K682ywcWJ3PASCcycH/22u+Bh7Aa0ehM2Fu4z0SAE81HF9RkB21c5bEn4Dzw+/qNOyXr3DCTQDMBOdhi4nAgiFDGCinIa2owCEChUwD8qzd03PG+qdW/4fDzjUMcE1ZpIAAAAASUVORK5CYII=');
  }

  div.callout-caution.callout-style-default .callout-title {
    background-color: #ffe5d0
  }

  </style>
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
<meta name="twitter:title" content="Analisi di dati di trascrittomica con R">
<meta name="twitter:description" content="Corso Revelo - Analisi di dati di trascrittomica con R">
<meta name="twitter:card" content="summary_large_image">
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">


<section id="TitleSlide" class="slide level2" data-menu-title="TitleSlide" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.3">
<h2></h2>
<style>
  @import url('https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Julius+Sans+One&family=Montserrat:wght@300&family=Rock+Salt&display=swap');
</style>
<div style="position: absolute; left: 180px; top: 200px; height: 525px; width: 1500px; background-color: #69b1e9; padding: 20px; padding-left: 50px; border-radius: 5px;">
<p><span style="font-size: 120px; font-weight: bold; line-height: 1em; margin: 0px; font-family: 'Architects Daughter', cursive;">Introduzione all’RNASeq</span></p>
<p><span style="font-size: 40px;font-weight: bold;">Introduzione all’analisi RNASeq in R</span></p>
<p><span style="font-size: 40px;font-weight: bold;">Dipartimento di Biomedicina e Prevenzione</span></p>
<p><br> <br></p>
<p><span style="font-size: 40px; font-weight: bold;">Marco Chiapello, Revelo Datalab</span></p>
<p><span style="font-size: 30px;font-weight: bold;">Nomber 2024</span></p>
</div>
<!-- #  {background-image="images/back001.jpg" background-size="cover" background-opacity="0.1"}

[Overview]{.f60 .p-span-center}

:::{.p-img-center}
![](images/pst.jpg){width=1000}
::: -->
</section>
<section id="section" class="title-slide slide level1 center" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1></h1>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>

<section>
<section id="section-1" class="title-slide slide level1 center" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1></h1>
<p><span class="tit p-span-center">Pre-processing and Quality Control</span></p>
</section>
<section id="section-2" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-3" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Raw reads: The Starting Point</span></p>
<h3 id="fasta-file">FASTA file</h3>
<div class="callout callout-important no-icon callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<p><strong>Definition</strong></p>
</div>
<div class="callout-content">
<p>The FASTA format is a text-based format for representing either nucleotide sequences or amino acid sequences. Both nucleotides and amino acids are represented using single-letter codes. The first line before the nucleotide/amino acid sequence contains the name of the sequence, preceded by the “&gt;” symbol.</p>
</div>
</div>
</div>
<p>File extension could be: fasta, fna, ffn, faa, fa, frn</p>
</section>
<section id="section-4" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Raw reads: The Starting Point</span></p>
<h3 id="fasta-file-1">FASTA file</h3>
<div class="callout callout-important no-icon callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<p><strong>Definition</strong></p>
</div>
<div class="callout-content">
<p>The FASTA format is a text-based format for representing either nucleotide sequences or amino acid sequences. Both nucleotides and amino acids are represented using single-letter codes. The first line before the nucleotide/amino acid sequence contains the name of the sequence, preceded by the “&gt;” symbol.</p>
</div>
</div>
</div>
<pre><code>&gt;NM_001404729.1 Oryza sativa ribulose bisphosphate carboxylase small chain A
CTCAACAGCACTGCTACTGGACATACTCTACTACTACTAGCCAGTAAGCTAGCTAACTAACTACGTGGCT
ATGGCCCCCACCGTGATGGCCTCCTCGGCCACCTCCGTGGCTCCATTCCAAGGGCTCAANNNNNNNNNNN</code></pre>
</section>
<section id="section-5" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Raw reads: The Starting Point</span></p>
<h3 id="fasta-file-2">FASTA file</h3>
<p>Multi-fasta file:</p>
<pre><code>&gt;MCHU - Calmodulin - Human, rabbit, bovine, rat, and chicken
MADQLTEEQIAEFKEAFSLFDKDGDGTITTKELGTVMRSLGQNPTEAELQDMINEVDADGNGTID
FPEFLTMMARKMKDTDSEEEIREAFRVFDKDGNGYISAAELRHVMTNLGEKLTDEEVDEMIREA
DIDGDGQVNYEEFVQMMTAK

&gt;NM_001404729.1 Oryza sativa ribulose bisphosphate carboxylase small chain A
CTCAACAGCACTGCTACTGGACATACTCTACTACTACTAGCCAGTAAGCTAGCTAACTAACTACGTGGCT
ATGGCCCCCACCGTGATGGCCTCCTCGGCCACCTCCGTGGCTCCATTCCAAGGGCTCAANNNNNNNNNNN
&gt;HF583486.1 Homo sapiens SOD1 gene for alternative protein SOD1, isolate 144496
ATGGATTCCATGTTCATGAGTTTGGAGATAATACAGCAGGCTGTACCAGTGCAGGTCCTCACTTTAATCC
TCTATCCAGAAAACACGGTGGGCCAAAGGATGAAGAGAGGCATGTTGGAGACTTGGGCAATGTGA</code></pre>
</section>
<section id="section-6" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Raw reads: The Starting Point</span></p>
<h3 id="fastq-format">FASTQ format</h3>
<div class="callout callout-important no-icon callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<p><strong>Definition</strong></p>
</div>
<div class="callout-content">
<p>The FASTQ format is a text-based format for storing both a biological sequence (usually nucleotide sequence) and its corresponding quality scores. Both the sequence letter and quality score are each encoded with a single ASCII character for brevity. &nbsp;</p>
</div>
</div>
</div>
</section>
<section id="section-7" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Raw reads: The Starting Point</span></p>
<h3 id="fastq-format-1">FASTQ format</h3>
<div class="callout callout-important no-icon callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<p><strong>Definition</strong></p>
</div>
<div class="callout-content">
<p>The FASTQ format is a text-based format for storing both a biological sequence (usually nucleotide sequence) and its corresponding quality scores. Both the sequence letter and quality score are each encoded with a single ASCII character for brevity. &nbsp;</p>
</div>
</div>
</div>
<div class="f30">
<p>A FASTQ file is identified by 4 lines:</p>
<ul>
<li>Line 1: begins with the “@” symbol and is followed by the sequence identifier (as in FASTA sequences)<br>
</li>
<li>Line 2: contains the nucleotide sequence<br>
</li>
<li>Line 3: begins with a “+” and may contain additional sequence descriptions<br>
</li>
<li>Line 4: contains the quality values for the individual nucleotides</li>
</ul>
</div>
</section>
<section id="section-8" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Raw reads: The Starting Point</span></p>
<h3 id="fastq-format-2">FASTQ format</h3>
<div class="callout callout-important no-icon callout-titled callout-style-default">
<div class="callout-body">
<div class="callout-title">
<p><strong>Definition</strong></p>
</div>
<div class="callout-content">
<p>The FASTQ format is a text-based format for storing both a biological sequence (usually nucleotide sequence) and its corresponding quality scores. Both the sequence letter and quality score are each encoded with a single ASCII character for brevity. &nbsp;</p>
</div>
</div>
</div>
<div class="f35">
<pre><code>@SEQ_ID
GATTTGGGGTTCAAAGCAGTATCGATCAAATAGTAAATCCATTTGTTCAACTCACAGTTT
+
!''*((((***+))%%%++)(%%%%).1***-+*''))**55CCF&gt;&gt;&gt;&gt;&gt;&gt;CCCCCCC65</code></pre>
</div>
</section>
<section id="section-9" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Raw reads: The Starting Point</span></p>
<h3 id="fastq-format-3">FASTQ format</h3>
<h4 id="what-does-quality-score-mean">What does “quality score” mean?</h4>
<div class="f20">
<p>The quality score is used to identify the <strong>probability of the correct identification of the corresponding nucleotide</strong>.</p>
</div>

<img data-src="images/phred1.png" class="r-stretch"><aside class="notes">
<ul>
<li>I numeri da 33 a 126 sono i numeri ASCII corrispondenti alla lettera che indicano</li>
<li>Score più alto osservato 40, quindi molti si fermano li</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="section-10" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Raw reads: The Starting Point</span></p>
<h3 id="fastq-format-4">FASTQ format</h3>
<h4 id="what-does-quality-score-mean-1">What does “quality score” mean?</h4>
<div class="f20">
<p>The quality score is used to identify the <strong>probability of the correct identification of the corresponding nucleotide</strong>.</p>
</div>
<div class="centerI">
<p><img data-src="images/phred2.png" width="900"></p>
</div>
</section>
<section id="section-11" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Raw reads: The Starting Point</span></p>
<h3 id="fastq-format-5">FASTQ format</h3>
<ul>
<li><p>FASTQ files can be very large, so in most cases you won’t be dealing with myFile.fastq, but with myFile.fastq.gz.</p></li>
<li><p>This means that the file is compressed, so it is not readable by a text editor without first being decompressed.</p></li>
</ul>
</section>
<section id="section-12" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-13" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Pre-processing Tools</span></p>
<h3 id="fastqc">FastQC</h3>
<div class="columns">
<div class="column f30" style="width:60%;">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>A widely used tool for quality assessment of raw sequencing data.</li>
</ul></li>
<li><strong>What it does:</strong>
<ul>
<li>Generates comprehensive reports with various quality metrics.</li>
<li>Visualizes data quality using interactive plots and graphs.</li>
</ul></li>
<li><strong>Key metrics:</strong>
<ul>
<li>Per base sequence quality</li>
<li>Per sequence quality scores</li>
<li>Sequence length distribution</li>
<li>GC content</li>
<li>Adapter content</li>
<li>Overrepresented sequences</li>
</ul></li>
</ul>
</div><div class="column" style="width:30%;">
<p><br> <br> <span class="p-spam.center"><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FastQC</a></span></p>
</div></div>
</section>
<section id="section-14" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-15" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Pre-processing Tools</span></p>
<h3 id="umi-tools-extract">UMI-tools extract</h3>
<div class="columns">
<div class="column f25" style="width:50%;">
<ul>
<li><strong>What are UMIs?</strong>
<ul>
<li>Unique Molecular Identifiers (UMIs) are short random sequences attached to each cDNA molecule during library preparation.</li>
<li>They act as “molecular barcodes” to identify unique molecules.</li>
</ul></li>
<li><strong>Why are UMIs important?</strong>
<ul>
<li>UMIs help to distinguish PCR duplicates from true biological duplicates.</li>
<li>This improves the accuracy of gene expression quantification, especially for low-abundance transcripts.</li>
</ul></li>
</ul>
</div><div class="column f25" style="width:50%;">
<ul>
<li><strong>What does <code>UMI-tools extract</code> do?</strong>
<ul>
<li>Identifies and extracts UMIs from sequencing reads.</li>
<li>Moves UMIs to the read name for downstream analysis.</li>
<li>Requires information about the UMI location in the read.</li>
</ul></li>
<li><strong>Benefits:</strong>
<ul>
<li>Improved accuracy in gene expression quantification.</li>
<li>Reduced bias due to PCR amplification.<br> <br> <span class="p-spam.center"><a href="https://umi-tools.readthedocs.io/en/latest/index.html">UMI-tools</a></span></li>
</ul></li>
</ul>
</div></div>
<div class="p-img-center">
<p><img data-src="images/umi.png" width="1000"></p>
</div>
</section>
<section id="section-16" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-17" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Pre-processing Tools</span></p>
<h3 id="fastp-trim-galore"><a href="https://github.com/OpenGene/fastp">FastP</a> &amp; <a href="https://github.com/FelixKrueger/TrimGalore">Trim Galore!</a></h3>
<div class="columns">
<div class="column f30" style="width:50%;">
<ul>
<li><strong>Purpose:</strong>
<ul>
<li>To clean up raw sequencing data and improve the accuracy of downstream analysis.</li>
</ul></li>
<li><strong>Key functions:</strong>
<ul>
<li><strong>Adapter trimming:</strong> Remove adapter sequences from reads.</li>
<li><strong>Quality trimming:</strong> Trim low-quality bases from the ends of reads.</li>
</ul></li>
<li><strong>FastP:</strong>
<ul>
<li>All-in-one tool with extensive pre-processing features.</li>
<li>Includes quality filtering, error correction, UMI handling, and more.</li>
</ul></li>
</ul>
</div><div class="column f30" style="width:50%;">
<ul>
<li><strong>Trim Galore!:</strong>
<ul>
<li>Wrapper script for <code>cutadapt</code> (adapter trimming) and often includes <code>FastQC</code> (quality assessment).</li>
<li>Focuses on adapter and quality trimming with optimized parameters.</li>
</ul></li>
<li><strong>Benefits:</strong>
<ul>
<li>Improved alignment accuracy.</li>
<li>Reduced false positives in variant calling and gene expression analysis.</li>
<li>Increased efficiency of downstream analysis.</li>
</ul></li>
</ul>
</div></div>
</section>
<section id="section-18" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-19" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Pre-processing Tools</span></p>
<h3 id="bbsplit">BBSplit</h3>
<div class="columns">
<div class="column f30" style="width:60%;">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>A tool for removing contaminant reads from sequencing data.</li>
</ul></li>
<li><strong>Types of contamination:</strong>
<ul>
<li>Microbial DNA or RNA</li>
<li>Host DNA (in the case of RNA-Seq from host-associated samples)</li>
<li>Other unwanted sequences</li>
</ul></li>
<li><strong>How it works:</strong>
<ul>
<li>Aligns reads to a database of contaminant sequences.</li>
<li>Removes reads that map to the contaminant database.</li>
</ul></li>
</ul>
</div><div class="column" style="width:30%;">
<p><br> <br> <span class="p-spam.center"><a href="https://sourceforge.net/projects/bbmap/">BBMap</a></span></p>
</div></div>
</section>
<section id="section-20" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Pre-processing Tools</span></p>
<h3 id="sortmerna">SortMeRNA</h3>
<div class="columns">
<div class="column f30" style="width:60%;">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>A tool for filtering ribosomal RNA (rRNA) reads from sequencing data.</li>
</ul></li>
<li><strong>Why is it important?</strong>
<ul>
<li>rRNA often makes up a large proportion of total RNA.</li>
<li>Removing rRNA enriches for mRNA and other non-coding RNAs of interest.</li>
</ul></li>
<li><strong>How it works:</strong>
<ul>
<li>Uses a database of rRNA sequences to identify and remove rRNA reads.</li>
<li>Can be used with both single-end and paired-end data.</li>
</ul></li>
</ul>
</div><div class="column" style="width:30%;">
<p><br> <br> <span class="p-spam.center"><a href="https://sortmerna.readthedocs.io/en/latest/">SortMeRNA</a></span></p>
</div></div>
</section></section>
<section>
<section id="section-21" class="title-slide slide level1 center" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1></h1>
<p><span class="tit p-span-center">Alignment and Quantification</span></p>
</section>
<section id="section-22" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-23" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Alignment and Quantification</span></p>
<h3 id="read-alignment">Read Alignment</h3>
<div class="columns">
<div class="column f30" style="width:50%;">
<ul>
<li><strong>Purpose:</strong>
<ul>
<li>To map sequencing reads to a reference genome or transcriptome.</li>
</ul></li>
<li><strong>Why is it important?</strong>
<ul>
<li>Identify the genomic origin of each read.</li>
<li>Determine which genes or transcripts are expressed.</li>
<li>Discover genetic variations (SNPs, indels).</li>
</ul></li>
</ul>
</div><div class="column f30" style="width:50%;">
<ul>
<li><strong>Challenges:</strong>
<ul>
<li>Reads may contain sequencing errors.</li>
<li>Reads may map to multiple locations (multi-mapping reads).</li>
<li>Genomes contain repetitive regions.</li>
</ul></li>
<li><strong>Output:</strong> A BAM (Binary Alignment/Map) file containing alignment information.</li>
</ul>
</div></div>
<div class="p-img-center">
<p><img data-src="images/align.png" width="1000"></p>
</div>
</section>
<section id="section-24" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Alignment and Quantification</span></p>
<h3 id="sambam-format-storing-alignment-information">SAM/BAM Format: Storing Alignment Information</h3>
<ul>
<li><strong>What it is:</strong>
<ul>
<li>The standard file format for storing read alignments.</li>
<li><strong>SAM (Sequence Alignment/Map):</strong> A text-based format.</li>
<li><strong>BAM (Binary Alignment/Map):</strong> A compressed binary version of SAM.</li>
</ul></li>
<li><strong>Why is it important?</strong>
<ul>
<li>Provides a standardized way to store alignment information.</li>
<li>Allows for efficient storage and retrieval of large alignment datasets.</li>
<li>Facilitates compatibility between different bioinformatics tools.</li>
</ul></li>
</ul>
</section>
<section id="section-25" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Alignment and Quantification</span></p>
<h3 id="sambam-format-storing-alignment-information-1">SAM/BAM Format: Storing Alignment Information</h3>
<ul>
<li><strong>Content:</strong>
<ul>
<li><strong>Header section</strong>: Contains metadata about the alignment (reference genome, aligner used, etc.).</li>
</ul></li>
</ul>
<div class="code">
<pre><code>@RG     ID:1    SM:C5926_BM_IonCode_0118
@PG     ID:samtools     PN:samtools    VN:1.16.1    CL:samtools view -H C5926_BM_IonCode_0118.reassembled.bam</code></pre>
</div>
<aside class="notes">
<ul>
<li><span class="citation" data-cites="RG">@RG</span> = read group
<ul>
<li>ID = Read group identifier</li>
<li>SM = Sample</li>
</ul></li>
<li><span class="citation" data-cites="PG">@PG</span> = Program
<ul>
<li>ID = Program record identifier</li>
<li>PN = Program name</li>
<li>VN = software version</li>
<li>Cl = command line</li>
</ul></li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="section-26" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Alignment and Quantification</span></p>
<h3 id="sambam-format-storing-alignment-information-2">SAM/BAM Format: Storing Alignment Information</h3>
<ul>
<li><strong>Alignment section</strong>: Each line represents a single read and its alignment to the reference.
<ul>
<li>Includes information about mapping position, quality scores, alignment flags, and more.</li>
</ul></li>
</ul>
<p><img data-src="images/bam2.png" class="absolute" style="top: 500px; left: 0px; width: 800px; height: auto; "></p>
<p><img data-src="images/bam1.png" class="absolute" style="top: 400px; right: 20px; width: 550px; height: auto; "></p>
<aside class="notes">
<ul>
<li><p>11 campi obbligatori</p></li>
<li><p>Metadati non obbligatori</p></li>
<li><p>Concise Idiosyncratic Gapped Alignment Report (CIGAR) string</p></li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="section-27" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-28" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Alignment and Quantification</span></p>
<h3 id="star-aligner"><a href="https://github.com/alexdobin/STAR">STAR Aligner</a></h3>
<div class="columns">
<div class="column f40" style="width:30%;">
<p><img data-src="images/alignment_STAR_step3.png" width="400"></p>
</div><div class="column f30" style="width:70%;">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>A widely used splice-aware aligner for RNA-Seq data.</li>
</ul></li>
<li><strong>Key features:</strong>
<ul>
<li>Fast and efficient alignment.</li>
<li>Accurate handling of spliced reads.</li>
<li>Can detect novel splice junctions.</li>
<li>Supports various RNA-Seq protocols.</li>
</ul></li>
<li><strong>Advantages:</strong>
<ul>
<li>High accuracy</li>
<li>Speed</li>
<li>Versatility</li>
</ul></li>
</ul>
</div></div>
</section>
<section id="section-29" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Alignment and Quantification</span></p>
<h3 id="hisat2"><a href="http://daehwankimlab.github.io/hisat2/">HISAT2</a></h3>
<div class="columns">
<div class="column f40" style="width:30%;">
<p><img data-src="images/alignment_STAR_step3.png" width="400"></p>
</div><div class="column f30" style="width:70%;">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>Another popular splice-aware aligner for RNA-Seq.</li>
</ul></li>
<li><strong>Key features:</strong>
<ul>
<li>Based on the Burrows-Wheeler transform.</li>
<li>Fast and memory-efficient.</li>
<li>Supports both DNA and RNA alignment.</li>
</ul></li>
<li><strong>Comparison to STAR:</strong>
<ul>
<li>May be slightly faster for some datasets.</li>
<li>May have slightly lower accuracy for certain types of reads.</li>
<li>Choice often depends on specific needs and preferences.</li>
</ul></li>
</ul>
</div></div>
</section>
<section id="section-30" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-31" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Alignment and Quantification</span></p>
<h3 id="umi-tools-dedup">UMI-tools dedup</h3>
<div class="columns">
<div class="column f40" style="width:30%;">
<p><img data-src="images/dedup.jpg" width="400"></p>
</div><div class="column f25" style="width:70%;">
<ul>
<li><strong>What it does:</strong>
<ul>
<li>Identifies and removes PCR duplicates from RNA-Seq data using Unique Molecular Identifiers (UMIs).</li>
<li>Groups reads with the same UMI and genomic mapping location.</li>
<li>Selects the read with the highest quality score as the representative read for each group.</li>
</ul></li>
<li><strong>Why is it important?</strong>
<ul>
<li>PCR amplification during library preparation can introduce duplicate reads.</li>
<li>These duplicates can bias gene expression estimates, especially for low-abundance transcripts.</li>
<li><code>UMI-tools dedup</code> helps to correct for this bias and improve accuracy.</li>
</ul></li>
<li><strong>Benefits:</strong>
<ul>
<li>More accurate gene expression quantification.</li>
<li>Reduced false positives in differential expression analysis.</li>
<li>Improved sensitivity for detecting rare transcripts.</li>
</ul></li>
</ul>
<p><span class="p-spam.center"><a href="https://umi-tools.readthedocs.io/en/latest/index.html">UMI-tools</a></span></p>
</div></div>
</section>
<section id="section-32" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-33" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Alignment and Quantification</span></p>
<h3 id="rsem"><a href="https://github.com/deweylab/RSEM">RSEM</a></h3>
<div class="columns">
<div class="column f40" style="width:50%;">
<p><br> <br></p>
<p><img data-src="images/RSEM.png" width="700"></p>
</div><div class="column f30" style="width:50%;">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>RSEM is a software package for estimating gene and isoform expression levels from RNA-Seq data.</li>
</ul></li>
<li><strong>Key features:</strong>
<ul>
<li>Uses a statistical model to estimate transcript abundance from aligned reads.</li>
<li>Can handle different types of RNA-Seq data (single-end, paired-end).</li>
<li>Provides various output formats (counts, TPM, FPKM).</li>
</ul></li>
</ul>
</div></div>
<aside class="notes">
<p>RPKM (Reads Per Kilobase Million) or FPKM (Fragments Per Kilobase Million). However, TPM (Transcripts Per Kilobase Million) is now becoming quite popular</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section></section>
<section>
<section id="section-34" class="title-slide slide level1 center" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1></h1>
<p><span class="tit p-span-center">Pseudo-aligners</span></p>
<p><span class="f50 p-span-center">A New Approach to Quantification</span></p>
</section>
<section id="section-35" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-36" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Pseudo-aligners</span></p>
<h3 id="fast-and-efficient-transcript-abundance-estimation">Fast and Efficient Transcript Abundance Estimation</h3>
<div class="columns">
<div class="column f30" style="width:60%;">
<ul>
<li><strong>What they are:</strong>
<ul>
<li>A class of RNA-Seq quantification tools that do not require exact alignment of reads to a reference genome.</li>
</ul></li>
<li><strong>How they work:</strong>
<ul>
<li>Use lightweight algorithms to assign reads to transcripts<br>
</li>
<li>Effectively “pseudo-align” reads to transcripts without determining their precise genomic location.</li>
</ul></li>
<li><strong>Advantages:</strong>
<ul>
<li><strong>Speed:</strong> Significantly faster than traditional alignment-based methods.</li>
<li><strong>Efficiency:</strong> Reduced computational resources required.</li>
<li><strong>Accuracy:</strong> Comparable or even superior accuracy for transcript quantification.</li>
</ul></li>
</ul>
</div><div class="column f30" style="width:40%;">
<ul>
<li><strong>Popular tools:</strong>
<ul>
<li>Salmon</li>
<li>Kallisto</li>
</ul></li>
<li><strong>Applications:</strong>
<ul>
<li>Gene expression profiling</li>
<li>Differential expression analysis</li>
<li>Transcript isoform identification</li>
</ul></li>
</ul>
</div></div>
</section>
<section id="section-37" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Pseudo-aligners</span></p>
<h3 id="kallisto-vs-salmon">Kallisto vs Salmon</h3>
<div class="f30">
<table class="caption-top">
<colgroup>
<col style="width: 30%">
<col style="width: 34%">
<col style="width: 35%">
</colgroup>
<thead>
<tr class="header">
<th>Feature</th>
<th><span class="f50">Kallisto</span> <img data-src="images/bear.jpg" width="100"></th>
<th><span class="f50">Salmon</span> <img data-src="images/salmon.png" width="190"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>Algorithm</strong></td>
<td>Pseudoalignment (k-mers)</td>
<td>Quasi-mapping (BWT)</td>
</tr>
<tr class="even">
<td><strong>Speed</strong></td>
<td>Fast</td>
<td>Faster</td>
</tr>
<tr class="odd">
<td><strong>Memory Usage</strong></td>
<td>Lower</td>
<td>Higher</td>
</tr>
<tr class="even">
<td><strong>Features</strong></td>
<td>Basic</td>
<td>More advanced (library types, bootstrapping)</td>
</tr>
<tr class="odd">
<td><strong>Accuracy</strong></td>
<td>High</td>
<td>High</td>
</tr>
<tr class="even">
<td><strong>Ease of Use</strong></td>
<td>High</td>
<td>High</td>
</tr>
<tr class="odd">
<td><strong>Output Formats</strong></td>
<td>Abundance estimates</td>
<td>Abundance estimates, counts, TPM, etc.</td>
</tr>
<tr class="even">
<td><strong>Applications</strong></td>
<td>Gene expression, DE analysis</td>
<td>Gene expression, DE analysis, isoform quantification</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="section-38" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Pseudo-aligners</span></p>
<h3 id="salmon-output-file-formats">Salmon Output File Formats</h3>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[40,-20,60]">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/salm1.png"></p>
<figcaption>Folder</figcaption>
</figure>
</div>
</div>
<div class="quarto-figure-spacer quarto-layout-cell" style="flex-basis: 16.7%;justify-content: flex-start;">
<p>&nbsp;</p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="images/salmon2.png"></p>
<figcaption>quant.sf</figcaption>
</figure>
</div>
</div>
</div>
</div>
<aside class="notes">
<ol type="1">
<li><p>Salmon’s main output is its quantification file =&gt; quant.sf</p></li>
<li><p>cmd_info.json. This is a JSON format file that records the main command line parameters with which Salmon was invoked</p></li>
<li><p>Auxiliary files: The top-level quantification directory will contain an auxiliary directory called aux_info</p>
<ul>
<li><p>The auxiliary directory will contain a JSON format file called meta_info.json which contains meta information about the run, including stats such as the number of observed and mapped fragments</p></li>
<li><p>If Salmon was run with automatic inference of the library type, information contained in this file is the inferred library type</p></li>
</ul></li>
<li><p>When run in mapping-based mode, the quantification directory will contain a file called lib_format_counts.json. This JSON file reports the number of fragments that had at least one mapping compatible with the designated library format</p></li>
</ol>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
<p>d</p>
</section></section>
<section>
<section id="section-39" class="title-slide slide level1 center" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1></h1>
<p><span class="tit p-span-center">Post-alignment Processing</span></p>
</section>
<section id="section-40" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-41" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Post-alignment Processing</span></p>
<h3 id="refining-your-alignments">Refining Your Alignments</h3>
<div class="f30">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>A series of steps performed after read alignment to improve data quality and prepare for downstream analysis.</li>
</ul></li>
<li><strong>Key steps:</strong>
<ul>
<li><strong>Deduplication:</strong> Removing PCR duplicates to avoid bias in gene expression estimates. (Blue line)</li>
<li><strong>Generating coverage tracks:</strong> Visualizing read distribution and assessing sequencing uniformity.</li>
<li><strong>(Optional) Other steps:</strong> Depending on the analysis goals, this may include filtering reads, recalibrating base quality scores, and other processing steps.</li>
</ul></li>
<li><strong>Why is it important?</strong>
<ul>
<li>Improves the accuracy and reliability of downstream analysis.</li>
<li>Reduces noise and bias in the data.</li>
<li>Facilitates data interpretation and visualization.</li>
</ul></li>
</ul>
</div>
</section>
<section id="section-42" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Post-alignment Processing</span></p>
<h3 id="samtools-manipulating-and-analyzing-alignments"><a href="https://samtools.github.io/">SAMtools</a>: Manipulating and Analyzing Alignments</h3>
<div class="f30">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>A suite of command-line utilities for working with SAM/BAM files.</li>
<li>A Swiss Army Knife for SAM/BAM Files</li>
</ul></li>
<li><strong>Key functions:</strong>
<ul>
<li><strong><code>samtools sort</code>:</strong> Sorts alignments in a BAM file by coordinate.
<ul>
<li>Why is sorting important? Many downstream tools require sorted BAM files for efficient processing.</li>
</ul></li>
<li><strong><code>samtools index</code>:</strong> Creates an index file for a sorted BAM file.
<ul>
<li>Why is indexing important? Allows for fast retrieval of specific regions of the alignment.</li>
</ul></li>
<li><strong><code>samtools stats</code>:</strong> Generates statistics about an alignment file.
<ul>
<li>Provides insights into alignment quality, read depth, and other metrics.</li>
</ul></li>
</ul></li>
</ul>
</div>
</section>
<section id="section-43" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Post-alignment Processing</span></p>
<h3 id="picard-markduplicates-deduplication-without-umis"><a href="https://broadinstitute.github.io/picard/">Picard</a> MarkDuplicates: Deduplication without UMIs</h3>
<div class="f40">
<ul>
<li><strong>How it works:</strong>
<ul>
<li>Identifies duplicates based on mapping coordinates of reads.</li>
<li>Marks duplicate reads in the BAM file.</li>
</ul></li>
<li><strong>Considerations:</strong>
<ul>
<li>Less accurate than UMI-based methods.</li>
<li>Can be used when UMIs are not available.</li>
<li>Often used in combination with other filtering steps.</li>
</ul></li>
</ul>
</div>
</section>
<section id="section-44" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Post-alignment Processing</span></p>
<h3 id="bedtools-genomecov-quantifying-read-depth"><a href="https://bedtools.readthedocs.io/en/latest/">BEDTools</a> genomecov: Quantifying Read Depth</h3>
<div class="f40">
<ul>
<li><strong>What it does:</strong>
<ul>
<li>Calculates the number of reads overlapping each position in the genome.</li>
<li>Outputs a coverage file in various formats (BEDGRAPH, bed, etc.).</li>
</ul></li>
<li><strong>Options:</strong>
<ul>
<li>Calculate coverage for different strand orientations.</li>
<li>Normalize coverage by library size.</li>
<li>Generate histograms of coverage depth.</li>
</ul></li>
</ul>
</div>
</section></section>
<section>
<section id="section-45" class="title-slide slide level1 center" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1></h1>
<p><span class="tit p-span-center">Downstream Analysis</span></p>
</section>
<section id="section-46" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section>
<section id="section-47" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Downstream Analysis</span></p>
<h3 id="rseqc-comprehensive-quality-assessment-of-your-rna-seq-data"><a href="https://rseqc.sourceforge.net">RSeQC</a>: Comprehensive Quality Assessment of Your RNA-Seq Data</h3>
<div class="f25">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>A suite of tools for RNA-Seq quality control and analysis.</li>
</ul></li>
<li><strong>Key functions:</strong>
<ul>
<li><strong>Read distribution:</strong> Assess the distribution of reads across different genomic features (e.g., coding regions, introns, UTRs).</li>
<li><strong>Gene body coverage:</strong> Evaluate the uniformity of coverage across gene bodies.</li>
<li><strong>Strand specificity:</strong> Check the strand specificity of the library.</li>
<li><strong>Junction saturation:</strong> Determine if sufficient sequencing depth has been achieved for junction detection.</li>
<li><strong>Other QC metrics:</strong> Assess rRNA contamination, insert size distribution, and more.</li>
</ul></li>
<li><strong>Benefits:</strong>
<ul>
<li>Provides comprehensive quality assessment of RNA-Seq data.</li>
<li>Identifies potential biases and issues in library preparation or sequencing.</li>
<li>Helps to ensure the reliability and accuracy of downstream analysis.</li>
</ul></li>
</ul>
</div>
</section>
<section id="section-48" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Downstream Analysis</span></p>
<h3 id="preseq-assessing-the-diversity-of-your-library"><a href="http://smithlabresearch.org/software/preseq/">Preseq</a>: Assessing the Diversity of Your Library</h3>
<div class="f25">
<ul>
<li><strong>What it is:</strong> A tool for estimating the complexity of a sequencing library.</li>
<li><strong>Why is library complexity important?</strong>
<ul>
<li>Library complexity refers to the number of unique DNA fragments in your library.</li>
<li>Higher complexity means a more diverse representation of the original RNA population.</li>
<li>Low complexity can lead to reduced sequencing efficiency and biased results.</li>
</ul></li>
<li><strong>What does Preseq do?</strong>
<ul>
<li>Uses statistical modeling to predict the number of unique reads that would be obtained with deeper sequencing.</li>
<li>Helps to assess whether additional sequencing is likely to yield new information.</li>
<li>Can be used to compare the complexity of different libraries.</li>
</ul></li>
<li><strong>Benefits:</strong>
<ul>
<li>Optimize sequencing depth.</li>
<li>Avoid unnecessary sequencing costs.</li>
<li>Improve the quality and efficiency of RNA-Seq experiments.</li>
</ul></li>
</ul>
</div>
</section>
<section id="section-49" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Downstream Analysis</span></p>
<h3 id="qualimap-rna-seq-a-deep-dive-into-your-rna-seq-data"><a href="http://qualimap.conesalab.org">Qualimap RNA-Seq</a>: A Deep Dive into Your RNA-Seq Data</h3>
<div class="f25">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>A comprehensive tool for quality control and analysis of RNA-Seq data.</li>
</ul></li>
<li><strong>Key features:</strong>
<ul>
<li><strong>Alignment quality:</strong> Assesses mapping quality, mismatch rates, and indel rates.</li>
<li><strong>Coverage analysis:</strong> Evaluates coverage uniformity across genes and transcripts.</li>
<li><strong>Junction analysis:</strong> Examines splice junctions and identifies non-canonical junctions.</li>
<li><strong>Transcript coverage:</strong> Assesses coverage of known transcripts and identifies potential novel transcripts.</li>
<li><strong>rRNA content:</strong> Estimates the proportion of rRNA reads in the data.</li>
<li><strong>5’ and 3’ bias:</strong> Detects biases in read coverage towards the 5’ or 3’ ends of transcripts.</li>
</ul></li>
<li><strong>Benefits:</strong>
<ul>
<li>Provides detailed insights into the quality of RNA-Seq alignments.</li>
<li>Identifies potential biases and technical artifacts.</li>
<li>Helps to ensure the reliability and accuracy of downstream analysis.</li>
</ul></li>
</ul>
</div>
</section>
<section id="section-50" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Downstream Analysis</span></p>
<h3 id="kraken2-identifying-microbial-communities-in-your-samples">Kraken2: Identifying Microbial Communities in Your Samples</h3>
<div class="columns">
<div class="column f40" style="width:40%;">
<p><img data-src="images/kraken.png" width="700"></p>
</div><div class="column f25" style="width:60%;">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>A fast and accurate tool for assigning taxonomic labels to sequencing reads.</li>
</ul></li>
<li><strong>Why is it useful for RNA-Seq?</strong>
<ul>
<li>RNA-Seq data can sometimes contain reads from microbial organisms (e.g., in microbiome studies, host-associated samples).</li>
<li>Kraken2 can identify and classify these microbial reads, providing insights into the microbial community present in the sample.</li>
</ul></li>
<li><strong>How it works:</strong>
<ul>
<li>Assigns taxonomic classifications to reads based on the best match.</li>
</ul></li>
<li><strong>Benefits:</strong>
<ul>
<li>Fast and efficient classification.</li>
<li>Accurate identification of microbial species.</li>
<li>Can be used with both DNA and RNA sequencing data.</li>
</ul></li>
</ul>
</div></div>
</section>
<section id="section-51" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Downstream Analysis</span></p>
<h3 id="differential-expression-finding-the-genes-that-matter">Differential Expression: Finding the Genes that Matter</h3>
<div class="f30">
<ul>
<li><strong>What it is:</strong>
<ul>
<li>Identifying genes that are expressed at significantly different levels between two or more conditions (e.g., treated vs.&nbsp;control, healthy vs.&nbsp;diseased).</li>
</ul></li>
<li><strong>Why is it important?</strong>
<ul>
<li>Uncover genes involved in specific biological processes or disease states.</li>
<li>Identify potential biomarkers or therapeutic targets.</li>
</ul></li>
<li><strong>Tools:</strong>
<ul>
<li><code>DESeq2</code>: A popular R package for differential expression analysis.</li>
<li><code>edgeR</code>: Another widely used R package.</li>
<li><code>limma</code>: A flexible package for linear modeling and differential expression analysis.</li>
</ul></li>
</ul>
</div>
</section>
<section id="section-52" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<p><span class="subtit">Downstream Analysis</span></p>
<h3 id="multiqc-your-one-stop-shop-for-qc-reports"><a href="https://seqera.io/examples/rna-seq/multiqc_report.html">MultiQC</a>: Your One-Stop Shop for QC Reports</h3>
<div class="f25">
<ul>
<li><strong>What it is:</strong> A tool that aggregates results from multiple bioinformatics tools into a single HTML report.</li>
<li><strong>Why is it useful for RNA-Seq?</strong>
<ul>
<li>RNA-Seq analysis involves multiple QC steps (FastQC, SortMeRNA, Trim Galore!, etc.).</li>
<li>MultiQC combines the results from these tools into a user-friendly report.</li>
<li>This saves time and simplifies the QC assessment process.</li>
</ul></li>
<li><strong>Key features:</strong>
<ul>
<li>Supports a wide range of bioinformatics tools.</li>
<li>Generates interactive plots and tables for easy visualization.</li>
<li>Highlights potential issues and inconsistencies in the data.</li>
<li>Facilitates comparison of QC metrics across multiple samples.</li>
</ul></li>
<li><strong>Benefits:</strong>
<ul>
<li>Saves time and effort in QC assessment.</li>
<li>Improves clarity and organization of QC results.</li>
<li>Facilitates data interpretation and decision-making.</li>
</ul></li>
</ul>
</div>
</section>
<section id="section-53" class="slide level2" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2></h2>
<div class="p-img-center">
<p><img data-src="images/nfcore.png" width="2000"></p>
</div>
</section></section>
<section id="section-54" class="title-slide slide level1 center" data-background-image="images/qmark.jpg" data-background-size="cover" data-background-opacity="0.7">
<h1></h1>
<div style="position: absolute; left: 980px; top: 450px;">
<p><span style="font-size: 130px; font-weight: bold; color: white">Domande?</span></p>
</div>

<div class="quarto-auto-generated-content">
<p><img src="images/revelo.png" class="slide-logo"></p>
<div class="footer footer-default">
<p><a href="https://revelo-rnaseq.github.io/2024_RNASeq_website">home page</a></p>
</div>
</div>
</section>
    </div>
  </div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="../site_libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="../site_libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="../site_libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="../site_libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="../site_libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="../site_libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="../site_libs/revealjs/plugin/notes/notes.js"></script>
  <script src="../site_libs/revealjs/plugin/search/search.js"></script>
  <script src="../site_libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="../site_libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': false,
'previewLinksAuto': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: true,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: false,

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'default',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'none',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1600,

        height: 900,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const onCopySuccess = function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      }
      const getTextToCopy = function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
        text: getTextToCopy
      });
      clipboard.on('success', onCopySuccess);
      if (window.document.getElementById('quarto-embedded-source-code-modal')) {
        // For code content inside modals, clipBoardJS needs to be initialized with a container option
        // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
        const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
          text: getTextToCopy,
          container: window.document.getElementById('quarto-embedded-source-code-modal')
        });
        clipboardModal.on('success', onCopySuccess);
      }
        var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
        var mailtoRegex = new RegExp(/^mailto:/);
          var filterRegex = new RegExp('/' + window.location.host + '/');
        var isInternal = (href) => {
            return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
        }
        // Inspect non-navigation links and adorn them if external
     	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
        for (var i=0; i<links.length; i++) {
          const link = links[i];
          if (!isInternal(link.href)) {
            // undo the damage that might have been done by quarto-nav.js in the case of
            // links that we want to consider external
            if (link.dataset.originalHref !== undefined) {
              link.href = link.dataset.originalHref;
            }
          }
        }
      function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
        const config = {
          allowHTML: true,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start',
        };
        if (contentFn) {
          config.content = contentFn;
        }
        if (onTriggerFn) {
          config.onTrigger = onTriggerFn;
        }
        if (onUntriggerFn) {
          config.onUntrigger = onUntriggerFn;
        }
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note) {
            return note.innerHTML;
          } else {
            return "";
          }
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>