<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>m3_rnaseq_intro – Analisi di dati di trascrittomica con R</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script src="../site_libs/quarto-contrib/roughnotation-0.5.1/rough-notation.iife.js"></script>
<script src="../site_libs/quarto-contrib/roughnotation-init-1.0.0/rough.js"></script>
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script src="../site_libs/quarto-diagram/mermaid.min.js"></script>
<script src="../site_libs/quarto-diagram/mermaid-init.js"></script>
<link href="../site_libs/quarto-diagram/mermaid.css" rel="stylesheet">


<meta name="twitter:title" content="Analisi di dati di trascrittomica con R">
<meta name="twitter:description" content="Corso Revelo - Analisi di dati di trascrittomica con R">
<meta name="twitter:card" content="summary_large_image">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../materials/images/revelo.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Analisi di dati di trascrittomica con R</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-slides" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Slides</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-slides">    
        <li>
    <a class="dropdown-item" href="../materials/M1_intro.html">
 <span class="dropdown-text">Modulo I: Introduzione al corso</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../materials/M2_RiepilogoR.html">
 <span class="dropdown-text">Modulo II: Riepilogo di R</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../materials/M3_RNASeq_Intro.html">
 <span class="dropdown-text">Modulo III: Introduzione all’RNASeq</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://phd-toolbox-course.github.io/2022_PhD_Toolbox_course/M4_RicercaRiproducibile.html#1">
 <span class="dropdown-text">Modulo IV: Ricerca Riproducibile</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../materials/M5_software_code.html">
 <span class="dropdown-text">Modulo V: Files and Software</span></a>
  </li>  
    </ul>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="../materials/risorse.html"> 
<span class="menu-text">Materiale aggiuntivo</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://revelotraining.slack.com"> 
<span class="menu-text">Slack chat</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://drive.google.com/drive/folders/1rfrM639jZcpSzGWYVPXALyWFbHOk92qX?usp=sharing"> 
<span class="menu-text">Cartella condivisa</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="https://revelodatalabs.com"> 
<span class="menu-text">Revelo DataLabs</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/mchiapello/202304_REVELO_Torvergata"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#TitleSlide" id="toc-TitleSlide" class="nav-link active" data-scroll-target="#TitleSlide"></a></li>
  <li><a href="#section" id="toc-section" class="nav-link" data-scroll-target="#section"></a>
  <ul class="collapse">
  <li><a href="#section-1" id="toc-section-1" class="nav-link" data-scroll-target="#section-1"></a></li>
  <li><a href="#section-2" id="toc-section-2" class="nav-link" data-scroll-target="#section-2"></a></li>
  <li><a href="#section-3" id="toc-section-3" class="nav-link" data-scroll-target="#section-3"></a></li>
  <li><a href="#section-4" id="toc-section-4" class="nav-link" data-scroll-target="#section-4"></a></li>
  <li><a href="#section-5" id="toc-section-5" class="nav-link" data-scroll-target="#section-5"></a></li>
  <li><a href="#section-6" id="toc-section-6" class="nav-link" data-scroll-target="#section-6"></a></li>
  <li><a href="#section-7" id="toc-section-7" class="nav-link" data-scroll-target="#section-7"></a></li>
  <li><a href="#section-8" id="toc-section-8" class="nav-link" data-scroll-target="#section-8"></a></li>
  <li><a href="#section-9" id="toc-section-9" class="nav-link" data-scroll-target="#section-9"></a></li>
  <li><a href="#section-10" id="toc-section-10" class="nav-link" data-scroll-target="#section-10"></a></li>
  <li><a href="#section-11" id="toc-section-11" class="nav-link" data-scroll-target="#section-11"></a></li>
  <li><a href="#section-12" id="toc-section-12" class="nav-link" data-scroll-target="#section-12"></a></li>
  <li><a href="#section-13" id="toc-section-13" class="nav-link" data-scroll-target="#section-13"></a></li>
  <li><a href="#section-14" id="toc-section-14" class="nav-link" data-scroll-target="#section-14"></a></li>
  </ul></li>
  <li><a href="#section-15" id="toc-section-15" class="nav-link" data-scroll-target="#section-15"></a>
  <ul class="collapse">
  <li><a href="#section-16" id="toc-section-16" class="nav-link" data-scroll-target="#section-16"></a></li>
  <li><a href="#section-17" id="toc-section-17" class="nav-link" data-scroll-target="#section-17"></a></li>
  <li><a href="#section-18" id="toc-section-18" class="nav-link" data-scroll-target="#section-18"></a></li>
  <li><a href="#section-19" id="toc-section-19" class="nav-link" data-scroll-target="#section-19"></a></li>
  <li><a href="#section-20" id="toc-section-20" class="nav-link" data-scroll-target="#section-20"></a></li>
  <li><a href="#section-21" id="toc-section-21" class="nav-link" data-scroll-target="#section-21"></a></li>
  <li><a href="#section-22" id="toc-section-22" class="nav-link" data-scroll-target="#section-22"></a></li>
  <li><a href="#section-23" id="toc-section-23" class="nav-link" data-scroll-target="#section-23"></a></li>
  <li><a href="#section-24" id="toc-section-24" class="nav-link" data-scroll-target="#section-24"></a></li>
  <li><a href="#section-25" id="toc-section-25" class="nav-link" data-scroll-target="#section-25"></a></li>
  <li><a href="#section-26" id="toc-section-26" class="nav-link" data-scroll-target="#section-26"></a></li>
  <li><a href="#section-27" id="toc-section-27" class="nav-link" data-scroll-target="#section-27"></a></li>
  <li><a href="#section-28" id="toc-section-28" class="nav-link" data-scroll-target="#section-28"></a></li>
  <li><a href="#section-29" id="toc-section-29" class="nav-link" data-scroll-target="#section-29"></a></li>
  <li><a href="#section-30" id="toc-section-30" class="nav-link" data-scroll-target="#section-30"></a></li>
  <li><a href="#section-31" id="toc-section-31" class="nav-link" data-scroll-target="#section-31"></a></li>
  <li><a href="#section-32" id="toc-section-32" class="nav-link" data-scroll-target="#section-32"></a></li>
  <li><a href="#section-33" id="toc-section-33" class="nav-link" data-scroll-target="#section-33"></a></li>
  <li><a href="#section-34" id="toc-section-34" class="nav-link" data-scroll-target="#section-34"></a></li>
  <li><a href="#section-35" id="toc-section-35" class="nav-link" data-scroll-target="#section-35"></a></li>
  <li><a href="#section-36" id="toc-section-36" class="nav-link" data-scroll-target="#section-36"></a></li>
  </ul></li>
  <li><a href="#section-37" id="toc-section-37" class="nav-link" data-scroll-target="#section-37"></a>
  <ul class="collapse">
  <li><a href="#section-38" id="toc-section-38" class="nav-link" data-scroll-target="#section-38"></a></li>
  <li><a href="#section-39" id="toc-section-39" class="nav-link" data-scroll-target="#section-39"></a></li>
  <li><a href="#section-40" id="toc-section-40" class="nav-link" data-scroll-target="#section-40"></a></li>
  <li><a href="#section-41" id="toc-section-41" class="nav-link" data-scroll-target="#section-41"></a></li>
  <li><a href="#section-42" id="toc-section-42" class="nav-link" data-scroll-target="#section-42"></a></li>
  <li><a href="#section-43" id="toc-section-43" class="nav-link" data-scroll-target="#section-43"></a></li>
  <li><a href="#section-44" id="toc-section-44" class="nav-link" data-scroll-target="#section-44"></a></li>
  <li><a href="#section-45" id="toc-section-45" class="nav-link" data-scroll-target="#section-45"></a></li>
  <li><a href="#section-46" id="toc-section-46" class="nav-link" data-scroll-target="#section-46"></a></li>
  <li><a href="#section-47" id="toc-section-47" class="nav-link" data-scroll-target="#section-47"></a></li>
  <li><a href="#section-48" id="toc-section-48" class="nav-link" data-scroll-target="#section-48"></a></li>
  <li><a href="#section-49" id="toc-section-49" class="nav-link" data-scroll-target="#section-49"></a></li>
  <li><a href="#section-50" id="toc-section-50" class="nav-link" data-scroll-target="#section-50"></a></li>
  <li><a href="#section-51" id="toc-section-51" class="nav-link" data-scroll-target="#section-51"></a></li>
  <li><a href="#section-52" id="toc-section-52" class="nav-link" data-scroll-target="#section-52"></a></li>
  <li><a href="#section-53" id="toc-section-53" class="nav-link" data-scroll-target="#section-53"></a></li>
  <li><a href="#section-54" id="toc-section-54" class="nav-link" data-scroll-target="#section-54"></a></li>
  <li><a href="#section-55" id="toc-section-55" class="nav-link" data-scroll-target="#section-55"></a></li>
  <li><a href="#section-56" id="toc-section-56" class="nav-link" data-scroll-target="#section-56"></a></li>
  <li><a href="#section-57" id="toc-section-57" class="nav-link" data-scroll-target="#section-57"></a></li>
  <li><a href="#section-58" id="toc-section-58" class="nav-link" data-scroll-target="#section-58"></a></li>
  </ul></li>
  <li><a href="#section-59" id="toc-section-59" class="nav-link" data-scroll-target="#section-59"></a>
  <ul class="collapse">
  <li><a href="#section-60" id="toc-section-60" class="nav-link" data-scroll-target="#section-60"></a></li>
  <li><a href="#section-61" id="toc-section-61" class="nav-link" data-scroll-target="#section-61"></a></li>
  <li><a href="#section-62" id="toc-section-62" class="nav-link" data-scroll-target="#section-62"></a></li>
  <li><a href="#section-63" id="toc-section-63" class="nav-link" data-scroll-target="#section-63"></a></li>
  <li><a href="#section-64" id="toc-section-64" class="nav-link" data-scroll-target="#section-64"></a></li>
  <li><a href="#section-65" id="toc-section-65" class="nav-link" data-scroll-target="#section-65"></a></li>
  <li><a href="#section-66" id="toc-section-66" class="nav-link" data-scroll-target="#section-66"></a></li>
  <li><a href="#section-67" id="toc-section-67" class="nav-link" data-scroll-target="#section-67"></a></li>
  <li><a href="#section-68" id="toc-section-68" class="nav-link" data-scroll-target="#section-68"></a></li>
  <li><a href="#section-69" id="toc-section-69" class="nav-link" data-scroll-target="#section-69"></a></li>
  <li><a href="#section-70" id="toc-section-70" class="nav-link" data-scroll-target="#section-70"></a></li>
  <li><a href="#section-71" id="toc-section-71" class="nav-link" data-scroll-target="#section-71"></a></li>
  <li><a href="#section-72" id="toc-section-72" class="nav-link" data-scroll-target="#section-72"></a></li>
  <li><a href="#section-73" id="toc-section-73" class="nav-link" data-scroll-target="#section-73"></a></li>
  <li><a href="#section-74" id="toc-section-74" class="nav-link" data-scroll-target="#section-74"></a></li>
  <li><a href="#section-75" id="toc-section-75" class="nav-link" data-scroll-target="#section-75"></a></li>
  <li><a href="#section-76" id="toc-section-76" class="nav-link" data-scroll-target="#section-76"></a></li>
  <li><a href="#section-77" id="toc-section-77" class="nav-link" data-scroll-target="#section-77"></a></li>
  <li><a href="#section-78" id="toc-section-78" class="nav-link" data-scroll-target="#section-78"></a></li>
  <li><a href="#section-79" id="toc-section-79" class="nav-link" data-scroll-target="#section-79"></a></li>
  <li><a href="#section-80" id="toc-section-80" class="nav-link" data-scroll-target="#section-80"></a></li>
  <li><a href="#section-81" id="toc-section-81" class="nav-link" data-scroll-target="#section-81"></a></li>
  <li><a href="#section-82" id="toc-section-82" class="nav-link" data-scroll-target="#section-82"></a></li>
  <li><a href="#section-83" id="toc-section-83" class="nav-link" data-scroll-target="#section-83"></a></li>
  <li><a href="#section-84" id="toc-section-84" class="nav-link" data-scroll-target="#section-84"></a></li>
  </ul></li>
  <li><a href="#section-85" id="toc-section-85" class="nav-link" data-scroll-target="#section-85"></a>
  <ul class="collapse">
  <li><a href="#section-86" id="toc-section-86" class="nav-link" data-scroll-target="#section-86"></a></li>
  <li><a href="#section-87" id="toc-section-87" class="nav-link" data-scroll-target="#section-87"></a></li>
  <li><a href="#section-88" id="toc-section-88" class="nav-link" data-scroll-target="#section-88"></a></li>
  <li><a href="#section-89" id="toc-section-89" class="nav-link" data-scroll-target="#section-89"></a></li>
  <li><a href="#section-90" id="toc-section-90" class="nav-link" data-scroll-target="#section-90"></a></li>
  </ul></li>
  <li><a href="#section-91" id="toc-section-91" class="nav-link" data-scroll-target="#section-91"></a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="M3_RNASeq_Intro.html"><i class="bi bi-file-slides"></i>RevealJS</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">




<section id="TitleSlide" class="level2" data-menu-title="TitleSlide" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.3">
<h2 data-menu-title="TitleSlide" data-background-image="images/back001.jpg" data-background-size="cover" data-background-opacity="0.3" class="anchored" data-anchor-id="TitleSlide"></h2>
<style>
  @import url('https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Julius+Sans+One&family=Montserrat:wght@300&family=Rock+Salt&display=swap');
</style>
<div style="position: absolute; left: 180px; top: 200px; height: 525px; width: 1500px; background-color: #69b1e9; padding: 20px; padding-left: 50px; border-radius: 5px;">
<p><span style="font-size: 120px; font-weight: bold; line-height: 1em; margin: 0px; font-family: 'Architects Daughter', cursive;">Introduzione all’RNASeq</span></p>
<p><span style="font-size: 40px;font-weight: bold;">Introduzione all’analisi RNASeq in R</span></p>
<p><span style="font-size: 40px;font-weight: bold;">Dipartimento di Biomedicina e Prevenzione</span></p>
<p><br> <br></p>
<p><span style="font-size: 40px; font-weight: bold;">Marco Chiapello, Revelo Datalab</span></p>
<p><span style="font-size: 30px;font-weight: bold;">2023-03-31</span></p>
</div>
</section>
<section id="section" class="level1" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1"></h1>
<p><span class="tit p-span-center">The history of sequencing</span></p>
<section id="section-1" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-1"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
</section>
<section id="section-2" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-2"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br> <br></p>
<div class="p-img-center">
<div>

</div>
<div class="cell quarto-layout-panel" data-layout-ncol="1">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<p><img src="M3_RNASeq_Intro_files/figure-html/unnamed-chunk-1-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</div>
</div>
</section>
<section id="section-3" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-3"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1953</span>: <span style="font-size: 70px">The structure of the DNA double helix was discovered</span></p>
<div class="p-img-center">
<p><img src="images/wcf.jpg" class="img-fluid"></p>
</div>
<div class="tcenter">
<p>James Watson, Francis Crick and Rosalind Franklin</p>
</div>
<div class="notes">
<p>The structure of the DNA double helix was discovered in 1953 by <strong>James Watson and Francis Crick</strong>.</p>
<p><strong>Rosalind Franklin</strong> was an English X-ray crystallographer who also <strong>contributed</strong> a great deal and was central to the understanding of the molecular structures of DNA.</p>
<p>Watson and Crick received a <strong>Nobel Prize in 1962</strong></p>
<p>Franklin had passed away before her work was fully appreciated, and so she is now often referred to as ‘the dark lady of DNA’.</p>
</div>
</section>
<section id="section-4" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-4"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1964</span>: <span style="font-size: 70px">Robert Holley, the first person to sequence a tRNA molecule</span></p>
<div class="p-img-center">
<p><img src="images/rwh.jpg" class="img-fluid" width="300"></p>
</div>
<div class="tcenter">
<p>Robert W. Holley</p>
</div>
<div class="notes">
<p><strong>Nobel Prize</strong> in Physiology or Medicine in <strong>1968</strong> (with Har Gobind Khorana and Marshall Warren Nirenberg) for describing the structure of alanine transfer RNA, linking DNA and protein synthesis.</p>
</div>
</section>
<section id="section-5" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-5"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1972</span>: <span style="font-size: 70px">Paul Berg developed the first technology that permitted the isolation of defined DNA fragments</span></p>
<div class="p-img-center">
<p><img src="images/pb.jpg" class="img-fluid" width="300"></p>
</div>
<div class="tcenter">
<p>Paul Berg</p>
</div>
<div class="notes">
<p>Morto il 15 febbraio 2023</p>
<p>leading to the <strong>development of modern genetic engineering</strong>. Before this, only phages or virus DNA was available for sequencing.</p>
<p><strong>Nobel Prize</strong> in Chemistry in 1980, along with Walter Gilbert and Frederick Sanger.</p>
<p>The award recognized their contributions to basic <strong>research involving nucleic acids</strong>, especially recombinant DNA.</p>
</div>
</section>
<section id="section-6" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-6"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1973</span>: <span style="font-size: 70px">Walter Gilbert published the first nucleotide sequence</span></p>
<div class="p-img-center">
<p><img src="images/wg.jpg" class="img-fluid" width="300"></p>
</div>
<div class="tcenter">
<p>Walter Gilbert</p>
</div>
<div class="notes">
<p>consisting of 24 base pairs of the DNA lac operator.</p>
</div>
</section>
<section id="section-7" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-7"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 40px">1973</span>: <span style="font-size: 50px">Walter Gilbert produced ‘DNA sequencing by chemical degradation’.</span></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 40px">1973</span>: <span style="font-size: 50px">Frederick Sanger was the first to sequence the complete DNA genome of a bacteriophage, called phi X174</span></p>
<div>

</div>
<div class="p-img-center quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<p><img src="images/wg.jpg" class="img-fluid" width="300"></p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<p><img src="images/fs.jpg" class="img-fluid" width="300"></p>
</div>
</div>
</div>
<div class="notes">
<p>Sanger developed ‘DNA sequencing with chain-terminating inhibitors’.</p>
<p>Frederick Sanger was the first to sequence the complete DNA genome of a bacteriophage, called phi X174. He also developed ‘DNA sequencing with chain-terminating inhibitors’.</p>
<p>Gilbert, Frederick Sanger and Paul Berg were awarded the 1980 Nobel Prize in Chemistry</p>
<p>Gilbert and Sanger were recognized for their pioneering work in devising methods for determining the sequence of nucleotides in a nucleic acid.</p>
</div>
</section>
<section id="section-8" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-8"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1983</span>: <span style="font-size: 70px">DNA amplification technique was developed</span></p>
<div class="p-img-center">
<p><img src="images/km.jpg" class="img-fluid" width="300"></p>
</div>
<div class="tcenter">
<p>Kary Mullis</p>
</div>
<div class="notes">
<p>1993 <strong>Nobel Prize</strong> for Chemistry in recognition of his role in the invention of the polymerase chain reaction (PCR) technique</p>
</div>
</section>
<section id="section-9" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-9"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1986</span>: <span style="font-size: 70px">Leroy Hood announced the invention of the first semi-automated DNA sequencing machine</span></p>
<div class="p-img-center">
<p><img src="images/lh.jpg" class="img-fluid" width="400"></p>
</div>
<div class="tcenter">
<p>Leroy Hood</p>
</div>
<div class="notes">
<p>Hood improved the existing Sanger method of enzymatic sequencing, which was becoming the laboratory standard</p>
<p>fluorescent dyes</p>
</div>
</section>
<section id="section-10" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-10"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1987</span>: <span style="font-size: 70px">Applied Biosystems produced the first automated sequencing machine, called ABI370.</span></p>
<div class="p-img-center">
<p><img src="images/373.jpg" class="img-fluid" width="600"></p>
</div>
<div class="tcenter">
<p>Applied Biosystems 370A Prototype Automated DNA Gene Sequencer,</p>
</div>
<div class="notes">
<p>This was a significant advance for several scientific research projects, including mapping the human genome.</p>
<p>The system relied on Sanger’s dideoxy sequencing method and fluorescent dyes</p>
</div>
</section>
<section id="section-11" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-11"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1990</span>: <span style="font-size: 70px">The Human Genome Project formally began</span></p>
<div class="p-img-center">
<p><img src="images/hp.jpg" class="img-fluid" width="600"></p>
</div>
<div class="tcenter">
<p>Early days: a DNA-sequencing lab in 1994</p>
</div>
<div class="notes">
<p>It was expected to take around 15 years.</p>
</div>
</section>
<section id="section-12" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-12"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1998</span>: <span style="font-size: 70px">‘Method of nucleic acid amplification’ was developed</span></p>
<div class="p-img-center">
<p><img src="images/impr.png" class="img-fluid" width="800"></p>
</div>
<div class="tcenter">
<p><span style="font-size: 30px">Patent: https://patents.google.com/patent/WO1998044151A1/en</span></p>
</div>
<div class="notes">
<p>This was one of the major milestones in developing NGS technologies.</p>
</div>
</section>
<section id="section-13" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-13"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">2000</span>: <span style="font-size: 70px">A ‘rough draft’ of the human genome was finished by the Human Genome Project</span></p>
<div class="p-img-center">
<p><img src="images/hgc.jpeg" class="img-fluid" width="700"></p>
</div>
<div class="tcenter">
<p><span style="font-size: 20px">Wadman, M. ‘Rough draft’ of human genome wins researchers’ backing. Nature 393, 399–400 (1998). https://doi.org/10.1038/30790</span></p>
</div>
<div class="notes">

</div>
</section>
<section id="section-14" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-14"></h2>
<p><span class="subtit">Key moments for DNA sequencing:</span></p>
<p><br></p>
<div class="p-img-center">
<p><img src="images/over.png" class="img-fluid"></p>
</div>
<div class="tcenter">
<p><span style="font-size: 30px">A timeline illustrating the milestones in major genome assembly achievements [Formenti et al., 2020]</span></p>
</div>
<div class="notes">
<p>The background colour indicates what type of sequencing was used: Red represents early sequencing methods, yellow is Sanger-based shotgun methods, green is NGS and blue represents third-generation sequencing.</p>
</div>
</section>
</section>
<section id="section-15" class="level1" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1"></h1>
<p><span class="tit p-span-center">History of NGS platforms</span></p>
<section id="section-16" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-16"></h2>
<p><span class="subtit">First generation NGS</span></p>
<div class="fragment fade-in" data-fragment-index="1">
<p>The technology was based on:</p>
<ul>
<li><p>the chain-termination method</p></li>
<li><p>the chain-degradation method</p></li>
</ul>
</div>
<div class="fragment fade-in" data-fragment-index="2">
<div class="p-img-center">
<div>

</div>
<div class="cell quarto-layout-panel" data-layout-ncol="1">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<p><img src="M3_RNASeq_Intro_files/figure-html/unnamed-chunk-2-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</div>
</div>
</div>
<div class="notes">
<ul>
<li><p>the chain-termination method developed by Sanger in 1975</p></li>
<li><p>the chain-degradation method invented by Gilbert in 1977</p></li>
</ul>
</div>
</section>
<section id="section-17" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-17"></h2>
<p><span class="subtit">Chain-degradation method</span></p>
<div class="p-img-center">
<p><img src="images/mg.png" class="img-fluid" width="500"></p>
</div>
<div class="notes">
<ul>
<li><p>Gilbert sequencing requires radioactive labeling at one 5′ end of the DNA fragment to be sequenced</p></li>
<li><p>Chemical treatment generates breaks at a small proportion of one or two of the four nucleotide bases in each of four reactions (G, A+G, C, C+T)</p></li>
<li><p>with the improvement of the chain-termination method (see below), Gilbert sequencing has fallen out</p></li>
</ul>
</div>
</section>
<section id="section-18" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-18"></h2>
<p><span class="subtit">Chain-termination method</span></p>
<p><br></p>
<div class="p-img-center">
<p><img src="images/sanger1.png" class="img-fluid" width="1500"></p>
</div>
<div class="notes">
<ul>
<li><p>Dideoxynucleotide triphosphates (ddNTPs) lack the 3’-OH group of dNTPs that is essential for polymerase-mediated strand elongation in a PCR</p></li>
<li><p>radioactive labels, which were unstable, posed a health hazard, and required separate gels for each of the four DNA bases</p></li>
<li><p>The synthesis of the complementary DNA strand starts at the specific priming site and ends with the incorporation of a chain-terminating ddNTP that is randomly introduced instead of its corresponding dNTP</p></li>
<li><p>When these fragments are separated on a suitable gel matrix the sequence information can be obtained from the migration order of the bands (bottom to top)</p></li>
</ul>
</div>
</section>
<section id="section-19" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-19"></h2>
<p><span class="subtit">Chain-termination method</span></p>
<p><br></p>
<div class="p-img-center">
<p><img src="images/sanger2.png" class="img-fluid" width="1500"></p>
</div>
<div class="notes">
<ul>
<li><p>Leroy Hood developed chemistry that used fluorescent dyes of different colors—one for each of the four DNA bases. This system of “color-coding” eliminated the need to run several reactions in overlapping gels</p></li>
<li><p>Hood integrated laser and computer technology, eliminating the tedious process of information-gathering by hand.</p></li>
<li><p>fluorescently labeled ddATP, ddCTP, ddGTP and ddUTP</p></li>
<li><p>Subsequent fluorescent visualization of the product reveals the incorporated nucleotide and hence the sequence</p></li>
</ul>
</div>
</section>
<section id="section-20" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-20"></h2>
<p><span class="subtit">Chain-termination method</span></p>
<div class="columns f40">
<div class="column" style="width:40%;">
<p><strong>Advantages</strong></p>
<ol type="1">
<li><p>Gold standard method for accurate detection of single nucleotide variants and small insertions/deletions</p></li>
<li><p>Cost effective where single samples need to be tested very urgently</p></li>
<li><p>Less reliant on computational tools than NGS</p></li>
<li><p>Longer fragments (up to approximately 1000bp) can be sequenced than in short read NGS</p></li>
</ol>
</div><div class="column" style="width:10%;">

</div><div class="column" style="width:40%;">
<p><strong>Limitations</strong></p>
<ol type="1">
<li><p>Limited throughput</p></li>
<li><p>Not cost effective for sequencing many genes in parallel</p></li>
<li><p>Can require a larger amount of input DNA than NGS</p></li>
<li><p>Sanger methods can only sequence short pieces of DNA–about 300 to 1000 base pairs.</p></li>
<li><p>The quality of a Sanger sequence is often not very good in the first 15 to 40 bases because that is where the primer binds.</p></li>
<li><p>Sequence quality degrades after 700 to 900 bases</p></li>
</ol>
</div>
</div>
</section>
<section id="section-21" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-21"></h2>
<p><span class="subtit">First generation NGS</span></p>
<p><br></p>
<p>The main advantages of first-generation NGS technologies are:</p>
<ul>
<li><p>Have a good overall sequence output</p></li>
<li><p>A high accuracy of 99.99%</p></li>
<li><p>Preparing nucleic acids of the ideal size is relatively easy</p></li>
</ul>
<div class="notes">
<p>Questi metodi non erano high througput!!</p>
</div>
</section>
<section id="section-22" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-22"></h2>
<p><span class="subtit">Second generation NGS</span></p>
<p><br></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p><span style="font-size: 50px; font-family: 'Julius Sans One', sans-serif">Second-generation NGS machines immediately began to drive the ‘genomics revolution’ by massively increased throughput by parallelizing many reactions</span></p>
</div>
</div>
</section>
<section id="section-23" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-23"></h2>
<p><span class="subtit">Second generation NGS</span></p>
<p><br></p>
<p>Second-generation sequencing platforms:</p>
<ol type="1">
<li><p>SOLiD: Sequencing by Oligonucleotide Ligation and Detection</p></li>
<li><p>454 GS FLX+: It uses pyrosequencing chemistry</p></li>
<li><p>NextSeq 550Dx: Sequence by synthesis</p></li>
</ol>
<div class="notes">
<ol type="1">
<li><p>SOLiD: Life Technologies in 2006</p></li>
<li><p>454: Roche in 2011</p></li>
<li><p>NextSeq 550Dx: Illumina in 2017</p></li>
</ol>
</div>
</section>
<section id="section-24" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-24"></h2>
<p><span class="subtit">Sequence by Ligation</span></p>
<p><br></p>
<div class="columns">
<div class="column" style="width:55%;">
<div class="r-stack p-img-center">
<p><img src="images/lig1.png" class="img-fluid" width="900"></p>
<p><img src="images/lig2.png" class="fragment img-fluid" width="900"></p>
</div>
</div><div class="column" style="width:45%;">
<p><br></p>
<div class="fragment">
<ul>
<li><p><svg aria-hidden="true" role="img" viewbox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:green;overflow:visible;position:relative;"><path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM232 344V280H168c-13.3 0-24-10.7-24-24s10.7-24 24-24h64V168c0-13.3 10.7-24 24-24s24 10.7 24 24v64h64c13.3 0 24 10.7 24 24s-10.7 24-24 24H280v64c0 13.3-10.7 24-24 24s-24-10.7-24-24z"></path></svg> Considered to be one of the most accurate second-generation sequencing technologies</p></li>
<li><p><svg aria-hidden="true" role="img" viewbox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:red;overflow:visible;position:relative;"><path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM184 232H328c13.3 0 24 10.7 24 24s-10.7 24-24 24H184c-13.3 0-24-10.7-24-24s10.7-24 24-24z"></path></svg> it can take up to seven days to complete a single run and its short read length of 35 bp</p></li>
<li><p><svg aria-hidden="true" role="img" viewbox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:red;overflow:visible;position:relative;"><path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM184 232H328c13.3 0 24 10.7 24 24s-10.7 24-24 24H184c-13.3 0-24-10.7-24-24s10.7-24 24-24z"></path></svg> Thermo Fisher Scientific shut down all SOLiD sequencing platforms in 2016</p></li>
</ul>
</div>
</div>
</div>
<div class="notes">
<ul>
<li><p>Fluorescently labeled molecules called 8-mers</p></li>
<li><p>8-mers are short ssDNA fragments where:</p>
<ul>
<li>the first and second nucleotides are the next two nucleotides of the elongating DNA strand<br>
</li>
<li>the third-to-fifth nucleotides are degenerate<br>
</li>
<li>the sixth to eighth nucleotides are inosine bases, where the eighth inosine base is attached to a characteristic fluorescent dye</li>
</ul></li>
</ul>
<p>A. The addition of primer and 8-mers, where if the first two nucleotides are complementary, DNA ligase ligates it to the primer.<br>
B. Unbound 8-mers are washed away, and a laser excites the fluorescent tag on the 8-mer to emit a detectable light signal that is captured<br>
C. A-B is repeated until the end of the DNA fragment.<br>
D. The newly synthesized DNA strand is melted off, and A-C is repeated using a primer with a length of N-1, N-2, and N-3</p>
</div>
</section>
<section id="section-25" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-25"></h2>
<p><span class="subtit">Pyrosequencing</span></p>
<div class="columns">
<div class="column" style="width:45%;">
<div class="p-img-center">
<p><img src="images/pyro.png" class="img-fluid" width="800"></p>
</div>
</div><div class="column" style="width:10%;">

</div><div class="column" style="width:45%;">
<p><br> <br></p>
<div class="fragment">
<ul>
<li><p><svg aria-hidden="true" role="img" viewbox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:green;overflow:visible;position:relative;"><path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM232 344V280H168c-13.3 0-24-10.7-24-24s10.7-24 24-24h64V168c0-13.3 10.7-24 24-24s24 10.7 24 24v64h64c13.3 0 24 10.7 24 24s-10.7 24-24 24H280v64c0 13.3-10.7 24-24 24s-24-10.7-24-24z"></path></svg> Large read lenght generation</p></li>
<li><p><svg aria-hidden="true" role="img" viewbox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:red;overflow:visible;position:relative;"><path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM184 232H328c13.3 0 24 10.7 24 24s-10.7 24-24 24H184c-13.3 0-24-10.7-24-24s10.7-24 24-24z"></path></svg> High reagent cost</p></li>
<li><p><svg aria-hidden="true" role="img" viewbox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:red;overflow:visible;position:relative;"><path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM184 232H328c13.3 0 24 10.7 24 24s-10.7 24-24 24H184c-13.3 0-24-10.7-24-24s10.7-24 24-24z"></path></svg> High error rate for homopolymers</p></li>
</ul>
</div>
</div>
</div>
<div class="notes">
<p>Durante il sequenziamento vengono, di volta in volta, aggiunti dei nucleotidi.</p>
<p>Se il nucleotide viene incorporato viene rilasciato un pirofosfato</p>
<p>Poi utilizzato come substrato dall’enzima ATP solforilasi.</p>
<p>Quest’ultimo enzima catalizza la conversione del pirofosfato in ATP.</p>
<p>L’ATP a sua volta è il cofattore utilizzato dalla luciferasi per catalizzare la reazione che coinvolge la luciferina e che porta alla produzione di luce.</p>
<p>La luce generata, solo in seguito alla incorporazione del nucleotide corretto,viene rilevata grazie ad un detector.</p>
</div>
</section>
<section id="section-26" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-26"></h2>
<p><span class="subtit">Sequence by synthesis</span></p>
</section>
<section id="section-27" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="1" class="anchored" data-anchor-id="section-27"></h2>
<div class="notes">
<p>Birthplace of high-throughput sequencing</p>
</div>
</section>
<section id="section-28" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-28"></h2>
<p><span class="subtit">Sequence by synthesis - History</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1997</span>: <span style="font-size: 70px">Evolution of a Novel Approach to Sequencing</span></p>
<div class="p-img-center">
<p><img src="images/bk.jpg" class="img-fluid" width="600"></p>
</div>
<div class="tcenter">
<p>Shankar Balasubramanian and David Klenerman</p>
</div>
<div class="notes">
<ul>
<li><p>Cambridge scientists Shankar Balasubramanian, Ph.D.&nbsp;and David Klenerman, Ph.D.&nbsp;were using fluorescently labeled nucleotides to observe the motion of a polymerase at the single molecule level as it synthesized DNA immobilized to a surface</p></li>
<li><p>during the summer of 1997 sparked ideas surrounding the use of clonal arrays and massively parallel sequencing of short reads using solid phase sequencing by reversible terminators</p></li>
<li><p>This technology was subsequently referred to as sequencing by synthesis technology, or SBS. This became the basis of an innovative DNA sequencing approach.</p></li>
</ul>
</div>
</section>
<section id="section-29" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-29"></h2>
<p><span class="subtit">Sequence by synthesis - History</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">1998</span>: <span style="font-size: 70px">Formation of Solexa</span></p>
<div class="p-img-center">
<p><img src="images/solexa.png" class="img-fluid" width="600"></p>
</div>
</section>
<section id="section-30" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-30"></h2>
<p><span class="subtit">Sequence by synthesis - History</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">2004</span>: <span style="font-size: 70px">Molecular Clustering Technology Integration</span></p>
<div class="p-img-center">
<p><img src="images/polony.jpg" class="img-fluid" width="800"></p>
</div>
<div class="tcenter">
<p>Cluster generation (also known as “bridge amplification”)</p>
</div>
<div class="notes">
<ul>
<li><p>Solexa acquired molecular clustering technology from Manteia.</p></li>
<li><p>The amplification of single DNA molecules into clusters enhanced the fidelity and accuracy of gene calling, while reducing the cost of the system optics through generation of a stronger signal.</p></li>
</ul>
</div>
</section>
<section id="section-31" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-31"></h2>
<p><span class="subtit">Sequence by synthesis - History</span></p>
<p><br></p>
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">2005</span>: <span style="font-size: 70px">phiX-174 Genome Sequencing</span></p>
<div class="fragment">
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">2005</span>: <span style="font-size: 70px">Integration of Lynx Therapeutics</span></p>
</div>
<div class="fragment">
<p><span style="font-family: 'Rock Salt', cursive;font-size: 60px">2007</span>: <span style="font-size: 70px">Illumina Acquires Solexa</span></p>
<div class="columns">
<div class="column" style="width:45%;">
<div class="p-img-center">
<p><img src="images/ga.jpg" class="img-fluid" width="500"></p>
</div>
</div><div class="column" style="width:10%;">

</div><div class="column" style="width:45%;">
<div class="p-img-center">
<p><img src="images/moore.png" class="img-fluid" width="500"></p>
</div>
</div>
</div>
</div>
<div class="notes">
<ul>
<li><p>The Solexa team sequenced the complete genome of the bacteriophage phiX-174, the same genome Sanger first sequenced using his method</p></li>
<li><p>However, the SBS technology generated significantly more sequencing data, delivering over 3 million bases from a single run.</p></li>
</ul>
<hr>
<ul>
<li>Solexa acquired instrumentation company Lynx Therapeutics, transforming the successful Solexa prototype into a commercial sequencing instrument: Genome Analyzer</li>
</ul>
<hr>
<ul>
<li><p>Solexa was acquired by Illumina</p></li>
<li><p>Next-generation sequencing (NGS) data output has increased at a rate that outpaces Moore’s law, more than doubling each year</p></li>
</ul>
</div>
</section>
<section id="section-32" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-32"></h2>
<p><span class="subtit">Sequence by synthesis - Process</span></p>
<div class="quarto-video"><iframe data-external="1" src="https://www.youtube.com/embed/fCd6B5HRaZ8" width="100%" height="85%" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
</section>
<section id="section-33" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-33"></h2>
<p><span class="subtit">Third generation NGS</span></p>
<p><br></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p><span style="font-size: 50px; font-family: 'Julius Sans One', sans-serif">Third-generation methods allow direct sequencing of single DNA molecules</span></p>
</div>
</div>
</section>
<section id="section-34" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-34"></h2>
<p><span class="subtit">Third generation NGS</span></p>
<p><br></p>
<p>Third-generation NGS platforms:</p>
<ol type="1">
<li><p>Single-molecule real-time sequencing</p></li>
<li><p>Nanopore sequencing</p></li>
</ol>
<div class="notes">
<ol type="1">
<li><p>Pacific Biosciences in 2011</p></li>
<li><p>Oxford Nanopore Technologies on 2014</p></li>
</ol>
</div>
</section>
<section id="section-35" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-35"></h2>
<p><span class="subtit">SMRS</span></p>
<div class="quarto-video"><iframe data-external="1" src="https://www.youtube.com/embed/_lD8JyAbwEo" width="100%" height="85%" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
</section>
<section id="section-36" class="level2" data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/pant.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-36"></h2>
<p><span class="subtit">Nanopore sequencing</span></p>
<div class="quarto-video"><iframe data-external="1" src="https://www.youtube.com/embed/CGWZvHIi3i0" width="100%" height="85%" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>
</section>
</section>
<section id="section-37" class="level1" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1"></h1>
<p><span class="tit p-span-center">RNASeq pipeline</span></p>
<section id="section-38" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-38"></h2>
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=6&gt;Isolate total RNA) --&gt; B(&lt;font size=6&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=6&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=6&gt;Sequencing) --&gt; E(&lt;font size=6&gt;Quantify expression)
      E --&gt; F(&lt;font size=6&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=6&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  class A,B,C,D,E,F,G className;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</section>
<section id="section-39" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-39"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class B,C,D,E,F,G className;
  class A classHigh;
  linkStyle 1 stroke:red
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Isolate total RNA</span></p>
<div class="fragment fade-in" data-fragment-index="1">
<div class="p-img-center">
<p><img src="images/rnaE.png" class="img-fluid" width="500"></p>
</div>
</div>
</div>
</div>
</section>
<section id="section-40" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-40"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,D,E,F,G className;
  linkStyle 0 stroke:red
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Quality control</span> <br></p>
<div class="incremental">
<ol class="incremental" type="1">
<li><p>RNA integrity is the major factor affecting the quality of sequencing data</p></li>
<li><p>Protocols of RNA-Seq libraries require samples with high-quality RNA</p></li>
<li><p>Prior to the construction of the libraries, it is necessary to assess RNA integrity number (RIN)</p></li>
<li><p>The RIN measurement is based on a machine learning algorithm</p></li>
<li><p>A higher RIN value indicates a higher degree of RNA integrity (range 1–10)</p></li>
<li><p><svg aria-hidden="true" role="img" viewbox="0 0 512 512" style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:red;overflow:visible;position:relative;"><path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM184 232H328c13.3 0 24 10.7 24 24s-10.7 24-24 24H184c-13.3 0-24-10.7-24-24s10.7-24 24-24z"></path></svg>RIN does not directly measure <strong>mRNA integrity</strong>, which is the main genetic material used in the construction of the libraries</p></li>
</ol>
</div>
</div>
</div>
<p>::::</p>
</section>
<section id="section-41" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-41"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,C,D,E,F,G className;
  class B classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Enrich a specific type of RNA</span></p>
<div class="incremental f40">
<ul class="incremental">
<li><p>mRNA</p></li>
<li><p>rRNAs and tRNAs (involved in mRNA translation)</p></li>
<li><p>Small nuclear RNAs (involved in splicing)</p></li>
<li><p>Small nucleolar RNAs (involved in the modification of rRNAs)</p></li>
<li><p>microRNA (regulate gene expression at the posttranscriptional level)</p></li>
<li><p>Long noncoding RNAs (chromatin remodelling, transcriptional control and posttranscriptional processing)</p></li>
</ul>
</div>
</div>
</div>
<p>::::</p>
<div class="notes">
<ul>
<li><p>mRNA molecules are the most studied RNA species because they encode proteins.</p></li>
<li><p>But recent advances in NGS technology have generated a focus on non-coding RNAs too, such as:</p></li>
</ul>
</div>
</section>
<section id="section-42" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-42"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,C,D,E,F,G className;
  class B classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Enrich a specific type of RNA</span></p>
<p><br></p>
<div class="incremental f40">
<p>Two options for mRNA enrichment</p>
<ul class="incremental">
<li><p><strong>mRNA enrichment</strong> – Selectively enriching for poly(A)-tailed transcripts</p></li>
<li><p><strong>RNA depletion</strong> – Selectively depleting abundant/off-target transcripts</p></li>
</ul>
</div>
</div>
</div>
<p>::::</p>
<div class="notes">
<p>3- RIN: has become a widely accepted standard for quality measurement</p>
</div>
</section>
<section id="section-43" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-43"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,D,E,F,G className;
  class C classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Prepare the RNA sequencing library</span></p>
<p><br></p>
<div>

</div>
<div class="cell quarto-layout-panel" data-layout-ncol="1">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<p><img src="M3_RNASeq_Intro_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="section-44" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-44"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,D,E,F,G className;
  class C classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Prepare the RNA sequencing library</span></p>
<p><br></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p><span style="font-size: 70px">A sequencing library is essentially a pool of RNA fragments with adapters attached</span></p>
</div>
</div>
</div>
</div>
</section>
<section id="section-45" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-45"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,D,E,F,G className;
  class C classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Prepare the RNA sequencing library</span> <span class="p-span-center f20">Step 1</span></p>
<p><br></p>
<p><strong>Fragmentation</strong></p>
<div class="{incremental}">
<ol type="1">
<li>Breaking of RNA strands into pieces</li>
</ol>
</div>
</div>
</div>
<div class="notes">
<p>1- which can be done by using physical, chemical or enzymatic methods</p>
</div>
</section>
<section id="section-46" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-46"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,D,E,F,G className;
  class C classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Prepare the RNA sequencing library</span> <span class="p-span-center f20">Step 2</span></p>
<p><br></p>
<p><strong>Attachment of adapters</strong></p>
<div class="incremental">
<ol class="incremental" type="1">
<li><p>Adapters are a short, chemically-synthesised oligonucleotide that can be attached to the ends of DNA molecules</p></li>
<li><p>Act as barcodes to identify where each nucleotide was originally located</p></li>
</ol>
</div>
<div class="fragment fade-in">
<div class="p-img-center">
<p><img src="images/adapter.png" class="img-fluid" width="900"></p>
</div>
</div>
</div>
</div>
<div class="notes">
<p>1- which can be done by using physical, chemical or enzymatic methods</p>
</div>
</section>
<section id="section-47" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-47"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,D,E,F,G className;
  class C classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Prepare the RNA sequencing library</span> <span class="p-span-center f20">Step 3</span></p>
<p><br></p>
<p><strong>Library quantification</strong></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p>This is key to obtaining high quality NGS data</p>
</div>
</div>
<div class="incremental">
<ol class="incremental" type="1">
<li><p>It provides the number of nucleic acids ready to be sequenced in a sample</p></li>
<li><p>It is important to try and obtain the highest complexity level as possible in an NGS library</p></li>
<li><p>Variety of techniques: UV absorption, intercalating dyes, hydrolysis probes and droplet digital emulsion PCR</p></li>
</ol>
</div>
</div>
</div>
<div class="notes">
<p>2- because this will reduce the amount of bias. Library complexity refers to the number of unique DNA fragments present – in other words, the library should reflect the starting material as much as possible. Reductions in complexity usually result from PCR amplification during library preparation, which elevates the number of duplicate reads</p>
</div>
</section>
<section id="section-48" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-48"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,E,F,G className;
  class D classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Sequencing</span> <span class="p-span-center f20">Step 1</span></p>
<p><br></p>
<p><strong>BCL to FASTQ conversion</strong></p>
<div class="incremental">
<ol class="incremental" type="1">
<li><p>Illumina sequencing technology uses cluster generation and sequencing by synthesis chemistry to sequence millions or billions of clusters on a flow cell</p></li>
<li><p>During sequencing, for each cluster, base calls are made and stored in the form of individual base call (or BCL) files</p></li>
<li><p>When sequencing completes, the base calls in the BCL files must be converted into sequence data</p></li>
</ol>
</div>
</div>
</div>
<div class="notes">
<p>3- This process is called BCL to FASTQ conversion</p>
</div>
</section>
<section id="section-49" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-49"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,E,F,G className;
  class D classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Sequencing</span> <span class="p-span-center f20">Step 1</span></p>
<p><br></p>
<p><strong>BCL to FASTQ conversion</strong></p>
<p><br></p>
<div class="p-img-center">
<p><img src="images/bc.png" class="img-fluid" width="800"></p>
</div>
</div>
</div>
</section>
<section id="section-50" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-50"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,E,F,G className;
  class D classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Sequencing</span> <span class="p-span-center f20">Step 2</span></p>
<p><br></p>
<p><strong>Pre-process</strong></p>
<p><br></p>
<div class="incremental">
<ol class="incremental" type="1">
<li><p>Assess the quality of sequencing data</p></li>
<li><p>Demultiplex by index or barcode</p></li>
<li><p>Remove adapter sequences</p></li>
<li><p>Trim reads by quality</p></li>
<li><p>Discard reads by quality/ambiguity</p></li>
</ol>
</div>
<div class="notes">
<p>2- (it is usually done in the sequencing facility)</p>
</div>
</div>
</div>
</section>
<section id="section-51" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-51"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,E,F,G className;
  class D classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Sequencing</span> <span class="p-span-center f20">Step 3</span></p>
<p><br></p>
<p><strong>Read mapping</strong></p>
<p><br></p>
<div class="incremental">
<ol class="incremental" type="1">
<li><p>Mapping reads to a reference genome or transcriptome</p></li>
<li><p>We need to retrieve a reference genome or transcriptome</p>
<ul class="incremental">
<li><p>Download from public databases</p></li>
<li><p>Create <em>De Novo</em></p></li>
</ul></li>
</ol>
</div>
</div>
</div>
<div class="notes">
<p>We are not exploring the <em>De Novo</em> option in this course</p>
</div>
</section>
<section id="section-52" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-52"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,E,F,G className;
  class D classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Sequencing</span> <span class="p-span-center f20">Step 4</span></p>
<p><br></p>
<p><strong>Read mapping</strong></p>
<p><br></p>
<div class="incremental">
<ol class="incremental" type="1">
<li><p>Fast (splice-unaware) aligners to a reference transcriptome</p></li>
<li><p>Splice-aware aligners to a reference genome</p></li>
<li><p>Quasi-mappers (alignment-free mappers) to a reference transcriptome</p></li>
</ol>
</div>
</div>
</div>
<div class="notes">
<ol type="1">
<li><p>Bowtie2, bwa, gem</p></li>
<li><p>STAR</p></li>
<li><p>Salmon, Kallisto and Sailfish</p></li>
</ol>
</div>
</section>
<section id="section-53" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-53"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,D,F,G className;
  class E classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Quantify expression</span></p>
<p><br></p>
<p><strong>Normalized expression units</strong></p>
<p><br></p>
<div class="incremental">
<ol class="incremental" type="1">
<li><p>Normalized gene expression units provide consistent and comparable measures to compare and visualize gene expression counts within and across samples</p></li>
<li><p>Normalized gene expression units are necessary to remove technical biases</p></li>
</ol>
</div>
</div>
</div>
<div class="notes">
<p>2- such as sequencing depth, RNA composition, and gene length in sequenced data</p>
<pre><code>- more sequencing depth produces more read count for a gene expressed at the same level

- differences in gene length generate unequal reads count for genes expressed at the same level (longer the gene more the read count)</code></pre>
</div>
</section>
<section id="section-54" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-54"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,D,F,G className;
  class E classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Quantify expression</span> <span class="p-span-center f20">Normalized expression units</span></p>
<p><br></p>
<p><strong>Reads per million</strong></p>
<p><br></p>
<div class="callout callout-style-default callout-tip callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Definition
</div>
</div>
<div class="callout-body-container callout-body">
<p>RPM (also known as CPM) is a basic gene expression unit that normalizes only for sequencing depth (depth-normalized counts)</p>
</div>
</div>
<div class="p-img-center">
<p><img src="images/rmp1.png" class="img-fluid" width="800"></p>
</div>
<p><br></p>
<div class="fragment fade-in" data-fragment-index="1">
<div class="p-img-center">
<p><img src="images/rmp2.png" class="img-fluid" width="800"></p>
</div>
</div>
</div>
</div>
<div class="notes">
<ul>
<li><p>RPM or CPM (Reads per million mapped reads or Counts per million mapped reads)</p></li>
<li><p>RPM is calculated by dividing the mapped reads count by a per million scaling factor of total mapped reads</p></li>
<li><p><strong>For example</strong>, You have sequenced one library with 5 million(M) reads. Among them, total 4 M matched to the genome sequence and 5000 reads matched to a given gene</p></li>
</ul>
</div>
</section>
<section id="section-55" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-55"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,D,F,G className;
  class E classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Quantify expression</span> <span class="p-span-center f20">Normalized expression units</span></p>
<p><br></p>
<p><strong>Reads per kilo base of transcript per million mapped reads</strong></p>
<p><br></p>
<div class="callout callout-style-default callout-tip callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Definition
</div>
</div>
<div class="callout-body-container callout-body">
<p>RPKM is normalized to correct the gene (transcript) lengths and library sizes (sequencing depth)</p>
</div>
</div>
<div class="p-img-center">
<p><img src="images/rpkm1.png" class="img-fluid" width="800"></p>
</div>
<p><br></p>
<div class="fragment fade-in" data-fragment-index="1">
<div class="p-img-center">
<p><img src="images/rpkm2.png" class="img-fluid" width="800"></p>
</div>
</div>
</div>
</div>
<div class="notes">
<ul>
<li><p>RPKM is most useful for comparing gene expression values within a sample, and it is not recommended for performing differential gene expression analysis (sample-to-sample comparisons)</p></li>
<li><p><strong>RPKM calculation example</strong>: You have sequenced one library with 5 M reads. Among them, total 4 M matched to the genome sequence and 5000 reads matched to a given gene with a length of 2000 bp</p></li>
</ul>
</div>
</section>
<section id="section-56" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-56"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,D,F,G className;
  class E classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Quantify expression</span> <span class="p-span-center f20">Normalized expression units</span></p>
<p><br></p>
<p><strong>Transcripts per million</strong></p>
<p><br></p>
<div class="callout callout-style-default callout-tip callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Definition
</div>
</div>
<div class="callout-body-container callout-body">
<p>TPM is a measurement of the proportion of transcripts in your pool of RNA</p>
</div>
</div>
<div class="p-img-center">
<p><img src="images/tpm1.png" class="img-fluid" width="800"></p>
</div>
<p><br></p>
<div class="fragment fade-in" data-fragment-index="1">
<div class="p-img-center">
<p><img src="images/tpm2.png" class="img-fluid" width="800"></p>
</div>
</div>
</div>
</div>
<div class="notes">
<ul>
<li><p>TPM is proportional to RPKM</p></li>
<li><p>In contrast to RPKM, the TPM average is constant and is proportional to the relative RNA molar concentration</p></li>
</ul>
</div>
</section>
<section id="section-57" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-57"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,D,E,G className;
  class F classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Differentially expression analysis</span></p>
<p><br></p>
<div class="f40 callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<ul>
<li><p>A basic task in the analysis of count data from RNA-seq is the detection of differentially expressed genes.</p></li>
<li><p>It determines genotypical differences between two or more conditions of cells, in support of specific hypothesis-driven studies.</p></li>
<li><p>The integration and the visualized representation of DGE result analysis functions can facilitate the downstream studies.</p></li>
</ul>
</div>
</div>
<div class="p-img-center">
<p><img src="images/dge1.png" class="img-fluid" width="400"></p>
</div>
</div>
</div>
<div class="notes">

</div>
</section>
<section id="section-58" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-58"></h2>
<div class="columns">
<div class="column" style="width:35%;">
<div class="cell" data-layout-align="default">
<div class="cell-output-display">
<div>
<p></p><figure class="figure"><p></p>
<div>
<pre class="mermaid mermaid-js">flowchart TB
  subgraph vitro [In vitro]
    direction TB
      A(&lt;font size=3&gt;Isolate total RNA) --&gt; B(&lt;font size=3&gt;Enrich a specific type of RNA)
      B --&gt; C(&lt;font size=3&gt;Prepare the RNA sequencing library)
  end
  subgraph silico [In silico]
    direction TB
      D(&lt;font size=3&gt;Sequencing) --&gt; E(&lt;font size=3&gt;Quantify expression)
      E --&gt; F(&lt;font size=3&gt;Differential Expression analysis)
      F --&gt; G(&lt;font size=3&gt;Biological conclusions)
  end
  vitro --&gt; silico
  classDef className fill:#D1D1D1,stroke:#333,stroke-width:1px
  classDef classHigh fill:#E3BE34,stroke:#333,stroke-width:4px
  class A,B,C,D,E,F className;
  class G classHigh;
</pre>
</div>
<p></p></figure><p></p>
</div>
</div>
</div>
</div><div class="column" style="width:65%;">
<p><span class="p-span-center">Biological conclusion</span></p>
<div class="p-img-center">
<p><img src="images/deg.png" class="img-fluid" width="900"></p>
</div>
</div>
</div>
<div class="notes">
<ul>
<li>To confirm the expression profile of genes in RNA-Seq results =&gt; Reverse transcription-quantitative PCR (RT-qPCR)</li>
</ul>
</div>
</section>
</section>
<section id="section-59" class="level1" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h1 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1"></h1>
<p><span class="tit p-span-center">How many samples do I need?</span></p>
<section id="section-60" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-60"></h2>
<div style="font-size: 80px; text-align: center">
<p>Power analysis</p>
</div>
</section>
<section id="section-61" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-61"></h2>
<div style="font-size: 80px; text-align: center">
<p>Power analysis</p>
</div>
<ul>
<li><strong>Type I error</strong>: controlled by the α value. Often set to 0.01 (1%) or 0.001 (0.1%) in RNA-seq experiments.</li>
</ul>
</section>
<section id="section-62" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-62"></h2>
<div style="font-size: 80px; text-align: center">
<p>Power analysis</p>
</div>
<ul>
<li><p><strong>Type I error</strong>: controlled by the α value. Often set to 0.01 (1%) or 0.001 (0.1%) in RNA-seq experiments.</p></li>
<li><p><strong>Type II error</strong>: controlled by the β value. (1−β) will give you the power of your analysis. Should be set to 70 or 80% to detect 70 or 80% of the differentially expressed genes. The number of biological replicates might be hard to reach in practice for RNA-seq experiments.</p></li>
</ul>
</section>
<section id="section-63" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-63"></h2>
<div style="font-size: 80px; text-align: center">
<p>Power analysis</p>
</div>
<ul>
<li><p><strong>Type I error</strong>: controlled by the α value. Often set to 0.01 (1%) or 0.001 (0.1%) in RNA-seq experiments.</p></li>
<li><p><strong>Type II error</strong>: controlled by the β value. (1−β) will give you the power of your analysis. Should be set to 70 or 80% to detect 70 or 80% of the differentially expressed genes. The number of biological replicates might be hard to reach in practice for RNA-seq experiments.</p></li>
<li><p><strong>Effect size</strong>: this is a parameter you will set. For instance, if you want to investigate genes that differ between treatments with a difference of their mean of 2 then the effect size is equal to 2.</p></li>
</ul>
</section>
<section id="section-64" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-64"></h2>
<div style="font-size: 80px; text-align: center">
<p>Power analysis</p>
</div>
<ul>
<li><p><strong>Type I error</strong>: controlled by the α value. Often set to 0.01 (1%) or 0.001 (0.1%) in RNA-seq experiments.</p></li>
<li><p><strong>Type II error</strong>: controlled by the β value. (1−β) will give you the power of your analysis. Should be set to 70 or 80% to detect 70 or 80% of the differentially expressed genes. The number of biological replicates might be hard to reach in practice for RNA-seq experiments.</p></li>
<li><p><strong>Effect size</strong>: this is a parameter you will set. For instance, if you want to investigate genes that differ between treatments with a difference of their mean of 2 then the effect size is equal to 2.</p></li>
<li><p><strong>Sample size</strong>: the quantity you want to calculate.</p></li>
</ul>
</section>
<section id="section-65" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-65"></h2>
<section id="lets-say-we-want" class="level4">
<h4 class="anchored" data-anchor-id="lets-say-we-want">Let’s say we want:</h4>
<ul>
<li>Type I error of 5%. (α=0.05)</li>
<li>Type II error of 0.2. (Power=1−β=0.8)</li>
<li>Effect size of 2. (d=2)</li>
</ul>
<div class="fragment fade-in">
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1"></a><span class="fu">library</span>(<span class="st">"pwr"</span>)</span>
<span id="cb2-2"><a href="#cb2-2"></a></span>
<span id="cb2-3"><a href="#cb2-3"></a><span class="fu">pwr.t.test</span>(<span class="at">d =</span> <span class="dv">2</span>,</span>
<span id="cb2-4"><a href="#cb2-4"></a>           <span class="at">power =</span> .<span class="dv">8</span>,</span>
<span id="cb2-5"><a href="#cb2-5"></a>           <span class="at">sig.level =</span> .<span class="dv">05</span>,</span>
<span id="cb2-6"><a href="#cb2-6"></a>           <span class="at">type =</span> <span class="st">"two.sample"</span>,</span>
<span id="cb2-7"><a href="#cb2-7"></a>           <span class="at">alternative =</span> <span class="st">"two.sided"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
     Two-sample t test power calculation 

              n = 5.089995
              d = 2
      sig.level = 0.05
          power = 0.8
    alternative = two.sided

NOTE: n is number in *each* group</code></pre>
</div>
</div>
</div>
</section>
</section>
<section id="section-66" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-66"></h2>
<section id="lets-say-we-want-1" class="level4">
<h4 class="anchored" data-anchor-id="lets-say-we-want-1">Let’s say we want:</h4>
<ul>
<li>Effect size of 1. (d=1)</li>
</ul>
<div class="fragment fade-in">
<div class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">"pwr"</span>)</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a><span class="fu">pwr.t.test</span>(<span class="at">d =</span> <span class="dv">1</span>,</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>           <span class="at">power =</span> .<span class="dv">8</span>,</span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>           <span class="at">sig.level =</span> .<span class="dv">05</span>,</span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>           <span class="at">type =</span> <span class="st">"two.sample"</span>,</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>           <span class="at">alternative =</span> <span class="st">"two.sided"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
     Two-sample t test power calculation 

              n = 16.71472
              d = 1
      sig.level = 0.05
          power = 0.8
    alternative = two.sided

NOTE: n is number in *each* group</code></pre>
</div>
</div>
</div>
</section>
</section>
<section id="section-67" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-67"></h2>
<p><span class="tit p-span-center">Power analysis for RNA-seq</span></p>
</section>
<section id="section-68" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-68"></h2>
<div style="font-size: 60px; text-align:center">
<p><strong>General Considerations</strong></p>
</div>
<p><br></p>
<div style="font-size: 60px">
<p>RNA-seq experiments often suffer from a low statistical power</p>
</div>
</section>
<section id="section-69" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-69"></h2>
<div style="font-size: 60px; text-align:center">
<p><strong>General Considerations</strong></p>
</div>
<p><br></p>
<div style="font-size: 60px">
<p>RNA-seq experiments often suffer from a low statistical power</p>
</div>
<p><br></p>
<div style="font-size: 60px">
<p>Low power can lead to a lack of reproducibility of the research findings</p>
</div>
</section>
<section id="section-70" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-70"></h2>
<div style="font-size: 60px; text-align:center">
<p><strong>General Considerations</strong></p>
</div>
<p><br></p>
<div style="font-size: 60px">
<p>RNA-seq experiments often suffer from a low statistical power</p>
</div>
<p><br></p>
<div style="font-size: 60px">
<p>Low power can lead to a lack of reproducibility of the research findings</p>
</div>
<p><br></p>
<div style="font-size: 60px">
<p>The number of replicates is one of the critical parameter related to the power of an analysis</p>
</div>
</section>
<section id="section-71" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-71"></h2>
<p><span class="tit p-span-center">Replicates</span></p>
</section>
<section id="section-72" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-72"></h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/image-259602676.png" class="img-fluid figure-img"></p>
<figcaption>Klaus B., EMBO J (2015) 34: 2727-2730</figcaption>
</figure>
</div>
<div class="notes">
<ul>
<li><p>Technical replicates: use the same biological sample to repeat the technical or experimental steps in order to accurately measure technical variation and remove it during analysis.</p></li>
<li><p>Biological replicates use different biological samples of the same condition to measure the biological variation between samples</p></li>
</ul>
</div>
</section>
<section id="section-73" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-73"></h2>
<p><span class="tit p-span-center">Do we need technical replicates?</span></p>
</section>
<section id="section-74" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-74"></h2>
<p><span class="tit p-span-center">No</span></p>
</section>
<section id="section-75" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-75"></h2>
<p><span class="tit p-span-center">No</span></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p><span class="f40">With the current RNA-Seq technologies, technical variation is much lower than biological variation and</span><span class="rn">technical replicates are unneccessary</span></p>
</div>
</div>
</section>
<section id="section-76" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-76"></h2>
<p><span class="tit p-span-center">Do we need biological replicates?</span></p>
</section>
<section id="section-77" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-77"></h2>
<p><span class="tit p-span-center">YES</span></p>
</section>
<section id="section-78" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-78"></h2>
<p><span class="tit p-span-center">YES</span></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p><span class="f40">Biological replicates are absolutely essential for <span class="rn">differential expression analysis</span></span></p>
</div>
</div>
</section>
<section id="section-79" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animate="true" class="anchored" data-anchor-id="section-79"></h2>
<p><span class="tit p-span-center">YES</span></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p><span class="f40">For differential expression analysis, the more biological replicates, the better the estimates of biological variation and the more precise our estimates of the mean expression levels</span></p>
</div>
</div>
</section>
<section id="section-80" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-80"></h2>
<div class="f40 tcenter fragment">
<p>Biological replicates are of greater importance than sequencing depth</p>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/image-154795781.png" height="700" class="figure-img"></p>
<figcaption>Liu, Y., et al., Bioinformatics (2014) 30(3): 301–304</figcaption>
</figure>
</div>
<div class="notes">
<p><span style="font-size: 50px; color: red">Cosa vedete? Come interpretate il grafico</span></p>
<p>Note that an increase in the number of replicates tends to return more DE genes than increasing the sequencing depth.</p>
</div>
</section>
<section id="section-81" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-81"></h2>
<div style="font-size: 60px; text-align: center">
<p><strong>Replicates are almost always preferred to greater sequencing depth for bulk RNA-Seq</strong></p>
</div>
<p><br></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p>However, guidelines depend on the experiment performed and the desired analysis.</p>
</div>
</div>
<div class="fragment fade-in" data-fragment-index="1">
<p>Here some examples:</p>
</div>
<div class="fragment fade-in" data-fragment-index="1">
<div style="font-size: 30px">
<p><strong>General gene-level differential expression</strong><br>
- ENCODE guidelines suggest 30 million SE reads per sample (stranded).<br>
- 15 million reads per sample is often sufficient, if there are a good number of replicates (&gt;3).<br>
- Spend money on more biological replicates, if possible.<br>
- Generally recommended to have read length &gt;= 50 bp</p>
</div>
</div>
</section>
<section id="section-82" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-82"></h2>
<div style="font-size: 60px; text-align: center">
<p><strong>Replicates are almost always preferred to greater sequencing depth for bulk RNA-Seq</strong></p>
</div>
<p><br></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p>However, guidelines depend on the experiment performed and the desired analysis.</p>
</div>
</div>
<p>Here some examples:</p>
<div style="font-size: 30px">
<p><strong>Gene-level differential expression with detection of lowly-expressed genes</strong><br>
- Similarly benefits from replicates more than sequencing depth.<br>
- Sequence deeper with at least 30-60 million reads depending on level of expression (start with 30 million with a good number of replicates). - Generally recommended to have read length &gt;= 50 bp</p>
</div>
</section>
<section id="section-83" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" class="anchored" data-anchor-id="section-83"></h2>
<div style="font-size: 60px; text-align: center">
<p><strong>Replicates are almost always preferred to greater sequencing depth for bulk RNA-Seq</strong></p>
</div>
<p><br></p>
<div class="callout callout-style-default callout-important callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Important
</div>
</div>
<div class="callout-body-container callout-body">
<p>However, guidelines depend on the experiment performed and the desired analysis.</p>
</div>
</div>
<p>Here some examples:</p>
<div style="font-size: 30px">
<p><strong>Isoform-level differential expression</strong><br>
- Of known isoforms, suggested to have a depth of at least 30 million reads per sample and paired-end reads.<br>
- Of novel isoforms should have more depth (&gt; 60 million reads per sample).<br>
- Choose biological replicates over paired/deeper sequencing.<br>
- Generally recommended to have read length &gt;= 50 bp, but longer is better as the reads will be more likely to cross exon junctions<br>
- Perform careful QC of RNA quality. Be careful to use high quality preparation methods and restrict analysis to high quality RIN # samples.</p>
</div>
</section>
<section id="section-84" class="level2" data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animtate="true">
<h2 data-background-image="images/stat001.jpg" data-background-size="cover" data-background-opacity="0.1" data-auto-animtate="true" class="anchored" data-anchor-id="section-84"></h2>
<div style="font-size: 60px; text-align: center">
<p><strong>Principles of a good experimental design</strong></p>
</div>
<div class="fragment fade-in" data-fragment-index="1">
<p><strong>Randomization</strong></p>
<p>This should remove undesired and sometimes unknown bias coming from an unidentified source of variation (e.g.&nbsp;different temperatures in the same greehouse).</p>
</div>
<div class="fragment fade-in" data-fragment-index="2">
<p><strong>Replication</strong></p>
<p>By repeating the same minimal experiment more than once, you can estimate the error due to the experimenter manipulation and your treatment effect.</p>
</div>
<div class="fragment fade-in" data-fragment-index="3">
<p><strong>Blocking</strong></p>
<p>Can help to reduce variability unexplained in one’s model. If your group consist of male and female individual, there is a chance that they will not respond in the same way to a given treatment. Therefore, “sex” should be a blocking factor in this case.</p>
</div>
</section>
</section>
<section id="section-85" class="level1" data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2">
<h1 data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2"></h1>
<p><span class="tit p-span-center">Applications</span></p>
<section id="section-86" class="level2" data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2">
<h2 data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2" class="anchored" data-anchor-id="section-86"></h2>
<div style="font-size: 80px; text-align: center">
<p>Diagnostics and disease profiling</p>
</div>
<p><br> <br></p>
<div class="fragment fade-down">
<div style="font-size: 60px">
<ol type="1">
<li><p>Transcriptional start sites</p></li>
<li><p>Uncovered alternative promoter usage</p></li>
<li><p>Novel splicing alterations</p></li>
</ol>
</div>
</div>
<div class="footer">
<p>Lowe et al., 2017</p>
</div>
<div class="notes">
<p>RNA-Seq approaches have allowed for the large-scale identification of</p>
</div>
</section>
<section id="section-87" class="level2" data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2">
<h2 data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2" class="anchored" data-anchor-id="section-87"></h2>
<div style="font-size: 80px; text-align: center">
<p>Human and pathogen transcriptomes</p>
</div>
<p><br> <br></p>
<div class="fragment fade-down">
<div style="font-size: 60px">
<ol type="1">
<li><p>Quantifying gene expression changes</p></li>
<li><p>Identifying novel virulence factors</p></li>
<li><p>Predicting antibiotic resistance</p></li>
<li><p>Unveiling host-pathogen immune interactions</p></li>
</ol>
</div>
</div>
<div class="footer">
<p>Lowe et al., 2017</p>
</div>
<div class="notes">
<p>A <strong>primary aim</strong> of this technology is to develop <strong>individualised treatment</strong></p>
<p>RNA-Seq of human pathogens has become an established method for</p>
</div>
</section>
<section id="section-88" class="level2" data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2">
<h2 data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2" class="anchored" data-anchor-id="section-88"></h2>
<div style="font-size: 80px; text-align: center">
<p>Responses to environment</p>
</div>
<p><br></p>
<div class="fragment fade-down">
<div style="font-size: 60px">
<ol type="1">
<li><p>Transcriptomics allows for the identification of genes and pathways that respond to biotic and abiotic environmental stresses</p></li>
<li><p>The nontargeted nature of transcriptomics allows for the identification of novel transcriptional networks in complex systems</p></li>
<li><p><span class="rn">RNA-virus identification for pathogen containment</span></p></li>
</ol>
</div>
</div>
<div class="footer">
<p>Lowe et al., 2017</p>
</div>
</section>
<section id="section-89" class="level2" data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2">
<h2 data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2" class="anchored" data-anchor-id="section-89"></h2>
<div style="font-size: 80px; text-align: center">
<p>Gene function annotation</p>
</div>
<div class="fragment fade-down">
<div style="font-size: 60px">
<ol type="1">
<li><p>All transcriptomic techniques have been particularly useful in identifying the functions of genes and identifying those responsible for particular phenotypes</p></li>
<li><p>Assembly of RNA-Seq reads is not dependent on a reference genome, and it is so ideal for gene expression studies of nonmodel organisms</p></li>
<li><p>RNA-Seq can also be used to identify previously unknown protein coding regions in existing sequenced genomes</p></li>
</ol>
</div>
</div>
<div class="footer">
<p>Lowe et al., 2017</p>
</div>
</section>
<section id="section-90" class="level2" data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2">
<h2 data-background-image="images/app001.jpg" data-background-size="cover" data-background-opacity="0.2" class="anchored" data-anchor-id="section-90"></h2>
<div style="font-size: 80px; text-align: center">
<p>Noncoding RNA</p>
</div>
<p><br></p>
<div class="fragment fade-down">
<div style="font-size: 60px">
<ol type="1">
<li><p>RNASeq is applicable to noncoding RNAs that are not translated into a protein, but instead, have direct functions</p></li>
<li><p>Many of these noncoding RNAs affect disease states, including cancer, cardiovascular, and neurological diseases</p></li>
</ol>
</div>
</div>
<div class="footer">
<p>Lowe et al., 2017</p>
</div>
<div class="notes">
<p>Transcriptomics is most commonly applied to the mRNA content of the cell, However</p>
</div>
</section>
</section>
<section id="section-91" class="level1" data-background-image="images/qmark.jpg" data-background-size="cover" data-background-opacity="0.7">
<h1 data-background-image="images/qmark.jpg" data-background-size="cover" data-background-opacity="0.7"></h1>
<div style="position: absolute; left: 980px; top: 450px;">
<p><span style="font-size: 130px; font-weight: bold; color: white">Domande?</span></p>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>This page was built with ❤️, <a href="https://cran.r-project.org">R</a> and <a href="https://quarto.org/">Quarto</a> by Marco Chiapello.</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>